% Template for PLoS
% Version 3.3 June 2016
%
% % % % % % % % % % % % % % % % % % % % % %
%
% -- IMPORTANT NOTE
%
% This template contains comments intended 
% to minimize problems and delays during our production 
% process. Please follow the template instructions
% whenever possible.
%
% % % % % % % % % % % % % % % % % % % % % % % 
%
% Once your paper is accepted for publication, 
% PLEASE REMOVE ALL TRACKED CHANGES in this file 
% and leave only the final text of your manuscript. 
% PLOS recommends the use of latexdiff to track changes during review, as this will help to maintain a clean tex file.
% Visit https://www.ctan.org/pkg/latexdiff?lang=en for info or contact us at latex@plos.org.
%
%
% There are no restrictions on package use within the LaTeX files except that 
% no packages listed in the template may be deleted.
%
% Please do not include colors or graphics in the text.
%
% The manuscript LaTeX source should be contained within a single file (do not use \input, \externaldocument, or similar commands).
%
% % % % % % % % % % % % % % % % % % % % % % %
%
% -- FIGURES AND TABLES
%
% Please include tables/figure captions directly after the paragraph where they are first cited in the text.
%
% DO NOT INCLUDE GRAPHICS IN YOUR MANUSCRIPT
% - Figures should be uploaded separately from your manuscript file. 
% - Figures generated using LaTeX should be extracted and removed from the PDF before submission. 
% - Figures containing multiple panels/subfigures must be combined into one image file before submission.
% For figure citations, please use "Fig" instead of "Figure".
% See http://journals.plos.org/plosone/s/figures for PLOS figure guidelines.
%
% Tables should be cell-based and may not contain:
% - spacing/line breaks within cells to alter layout or alignment
% - do not nest tabular environments (no tabular environments within tabular environments)
% - no graphics or colored text (cell background color/shading OK)
% See http://journals.plos.org/plosone/s/tables for table guidelines.
%
% For tables that exceed the width of the text column, use the adjustwidth environment as illustrated in the example table in text below.
%
% % % % % % % % % % % % % % % % % % % % % % % %
%
% -- EQUATIONS, MATH SYMBOLS, SUBSCRIPTS, AND SUPERSCRIPTS
%
% IMPORTANT
% Below are a few tips to help format your equations and other special characters according to our specifications. For more tips to help reduce the possibility of formatting errors during conversion, please see our LaTeX guidelines at http://journals.plos.org/plosone/s/latex
%
% For inline equations, please be sure to include all portions of an equation in the math environment.  For example, x$^2$ is incorrect; this should be formatted as $x^2$ (or $\mathrm{x}^2$ if the romanized font is desired).
%
% Do not include text that is not math in the math environment. For example, CO2 should be written as CO\textsubscript{2} instead of CO$_2$.
%
% Please add line breaks to long display equations when possible in order to fit size of the column. 
%
% For inline equations, please do not include punctuation (commas, etc) within the math environment unless this is part of the equation.
%
% When adding superscript or subscripts outside of brackets/braces, please group using {}.  For example, change "[U(D,E,\gamma)]^2" to "{[U(D,E,\gamma)]}^2". 
%
% Do not use \cal for caligraphic font.  Instead, use \mathcal{}
%
% % % % % % % % % % % % % % % % % % % % % % % % 
%
% Please contact latex@plos.org with any questions.
%
% % % % % % % % % % % % % % % % % % % % % % % %

\documentclass[10pt,letterpaper]{article}
\usepackage[top=0.85in,left=2.75in,footskip=0.75in]{geometry}

% amsmath and amssymb packages, useful for mathematical formulas and symbols
\usepackage{amsmath,amssymb}

% Use adjustwidth environment to exceed column width (see example table in text)
\usepackage{changepage}

% Use Unicode characters when possible
\usepackage[utf8x]{inputenc}

% textcomp package and marvosym package for additional characters
\usepackage{textcomp,marvosym}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

% makes the \textsubscript work on my older version of latex
\usepackage{fixltx2e}

% Use nameref to cite supporting information files (see Supporting Information section for more info)
\usepackage{nameref,hyperref}

% line numbers
\usepackage[right]{lineno}

% ligatures disabled
\usepackage{microtype}
\DisableLigatures[f]{encoding = *, family = * }

% color can be used to apply background shading to table cells only
\usepackage[table]{xcolor}
\usepackage{multirow} %connecting columns in tables
\usepackage{multicol}
\usepackage{longtable}

% array package and thick rules for tables
\usepackage{array}

% create "+" rule type for thick vertical lines
\newcolumntype{+}{!{\vrule width 2pt}}

% create \thickcline for thick horizontal lines of variable length
\newlength\savedwidth
\newcommand\thickcline[1]{%
  \noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
  \cline{#1}%
  \noalign{\vskip\arrayrulewidth}%
  \noalign{\global\arrayrulewidth\savedwidth}%
}

% \thickhline command for thick horizontal lines that span the table
\newcommand\thickhline{\noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
\hline
\noalign{\global\arrayrulewidth\savedwidth}}


% Remove comment for double spacing
%\usepackage{setspace} 
%\doublespacing

% Text layout
\raggedright
\setlength{\parindent}{0.5cm}
\textwidth 5.25in 
\textheight 8.75in

% Bold the 'Figure #' in the caption and separate it from the title/caption with a period
% Captions will be left justified
\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}
\renewcommand{\figurename}{Fig}

% Use the PLoS provided BiBTeX style
\bibliographystyle{plos2015}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother

% Leave date blank
\date{}

% Header and Footer with logo
\usepackage{lastpage,fancyhdr,graphicx}
\usepackage{epstopdf}
\pagestyle{myheadings}
\pagestyle{fancy}
\fancyhf{}
\setlength{\headheight}{27.023pt}
\lhead{\includegraphics[width=2.0in]{PLOS-submission.eps}}
\rfoot{\thepage/\pageref{LastPage}}
\renewcommand{\footrule}{\hrule height 2pt \vspace{2mm}}
\fancyheadoffset[L]{2.25in}
\fancyfootoffset[L]{2.25in}
\lfoot{\sf PLOS}

%% Include all macros below

\newcommand{\lorem}{{\bf LOREM}}
\newcommand{\ipsum}{{\bf IPSUM}}

%% END MACROS SECTION


\begin{document}
\vspace*{0.2in}

% Title must be 250 characters or less.
\begin{flushleft}
{\Large
\textbf\newline{Eco-evolutionary Dynamics on the Invasion Front Drives the Co-existence of Target Site and Quantitative Resistance} % Please use "title case" (capitalize all terms in the title except conjunctions, prepositions, and articles).
}
\newline
% Insert author names, affiliations and corresponding author email (do not include titles, positions, or degrees).
\\
Shaun Coutts\textsuperscript{1*},
Rob Freckelton\textsuperscript{1},
Helen Hicks\textsuperscript{1},
Paul Neve\textsuperscript{2},
Dylan Childs\textsuperscript{1},
\\
\bigskip
\textbf{1} Animal and Plant Sciences, University of Sheffield, Sheffield, UK
\\
\textbf{2} Affiliation Dept/Program/Center, Institution Name, City, State, Country
\\
\bigskip

% Insert additional author notes using the symbols described below. Insert symbol callouts after author names as necessary.
% 
% Remove or comment out the author notes below if they aren't used.
%
% Primary Equal Contribution Note
%\Yinyang These authors contributed equally to this work.

% Additional Equal Contribution Note
% Also use this double-dagger symbol for special authorship notes, such as senior authorship.
%\ddag These authors also contributed equally to this work.

% Current address notes
%\textcurrency Current Address: Dept/Program/Center, Institution Name, City, State, Country % change symbol to "\textcurrency a" if more than one current address note
% \textcurrency b Insert second current address 
% \textcurrency c Insert third current address

% Deceased author note
%\dag Deceased

% Group/Consortium Author Note
%\textpilcrow Membership list can be found in the Acknowledgments section.

% Use the asterisk to denote corresponding authorship and provide email address in note below.
* shaun.coutts@gmail.com

\end{flushleft}
% Please keep the abstract below 300 words
\section*{Abstract}
Evolved resistance threatens the use of important antibiotics, pesticides and herbicides (collectively xenobiotics). Xenobiotic resistance is a complex trait, controlled by a variable number of genes. At one extreme target site resistance is conferred by a single mutation, and at the other extreme quantitative resistance is conferred by the small additive effects of many mutations. There is growing evidence that target site and quantitative resistance exist within the same population. This has important implications for understanding how traits evolve, since target site resistance should displace the less effective more costly quantitative resistance. There are also important implications for the management of resistance as strategies that slow the evolution of target site resistance, such as cycling compounds, may increase selective pressure for quantitative resistance. We model resistance with both target site and quantitative resistance acting in the same population. We apply this model to the economically important weed, \textit{Alopecurus myosuroides} (Huds.). The ecological context determined the which form of resistance evolved. When target site resistance was introduced to an empty landscape it quickly reached fixation in the population. When target site resistance was introduced into a population with prior selection for quantitative resistance its evolution was slower. We also show that spatial structure in both target site and quantitative resistance can either promote or reduce the evolution of target site resistance, depending on the context. Because of this context dependence we observed the full range of outcomes under ecologically realistic scenarios and management relevant time scales; from target site resistance approaching fixation, to all resistance being quantitative, to both target site and quantitative resistance co-existing in the population. Given this strong context dependence we should not be surprised to see both target site and quantitative resistance existing in the same population.     

% Please keep the Author Summary between 150 and 200 words
\section*{Author Summary}
Growing resistance to antibiotics, pesticides and herbicides threaten food security and human health worldwide. Herbicide resistance alone threatens our ability to control weeds, which left unchecked can reduce yields of vital crops by 30\%. Further, resistance can evolve within three years, which means developing new compounds is not an effective solution on its own, we must also prolong the useful life of existing compounds. Complicating this effort is the fact that resistance can be conferred by a highly variable number of mutations, each providing very different levels of resistance. Strategies to control resistance conferred by a single, highly effective mutation (known as target site resistance), such as cycling compounds, can speed up the evolution of resistance conferred by many mutations, each with small effect (known as quantitative resistance), by increasing selection pressure. We model the evolution of both target site and quantitative resistance in the same population and show that the ecological context under which resistance evolves can lead to mainly target site resistance, or high levels of quantitative resistance, or both co-existing. This could lead to the worst of both worlds in terms of managing resistance, leaving few viable strategies to slow the evolution of resistance.      

\linenumbers

% Use "Eq" instead of "Equation" for equation citations.
\section*{Introduction}
There is currently a global resistance crisis \cite{Serv2013, Ross2014} and important antibiotics, pesticides and herbicides (collectively xenobiotics) are losing their efficacy \cite{Palu2001}. Together, these chemical tools are a crucial part of the world's food productions system \cite{Duke2012}, and are used worldwide to control life threatening infectious diseases. However, evolving resistance threatens their continued use \cite{Barb2011, Nkya2013}. Economic and regulatory conditions mean that bringing new, safe and effective compounds to market is time consuming and expensive \cite{Duke2012}. In addition, the useful life of new xenobiotics can be as short as three years if their use is not well managed \cite{Palu2001, Duke2012}. To address this crisis and reduce its impact we need to understand the factors that promote or constrain the evolution of resistance. 

Xenobiotic resistance is a complex trait, with a variable number of contributing loci \cite{Warw1991, Neve2007, Dely2013, Bauc2016}, complicating efforts to understand and manage its evolution. The best understood form of resistance is target site resistance \cite{Warw1991, Neve2007, Dely2013, Serv2013}, which is often controlled by mutation at a single locus \cite{Bauc2016}. This mutation alters the binding site of the xenobiotic so that it can no longer bind \cite{Dely2013, Bauc2016}. Target site resistance often confers very high levels of resistance, and incurs few fitness costs (i.e. reductions in survival, growth and/or fecundity)\cite{Warw1994, Vila2005, Bauc2016}. At the other extreme, quantitative resistance is conferred by the small effects of many genes \cite{Land1989, Mack2009, Dely2013, Rajo2013}. Quantitative resistance is usually achieved by metabolizing the xenobiotic, or transporting it away from its binding site \cite{Dely2013, Bauc2016} and often confers lower levels of resistance and incurs fitness costs \cite{Vila2005, Bauc2016}. There is also growing evidence that target site and quantitative resistance exist within the same population in both plants \cite{Warw1991, Vila2005, Herr2014, Han2016} and insects \cite{Gard1998, Donn2009, Bing2011, Hend2013, Oake2013}.

Whether resistance is conferred by target site or quantitative resistance has important implications for its management. For example using lower doses delays the evolution of target site resistance once present in a population, but may accelerate the evolution of quantitative resistance \cite{Gard1998}.
In addition the cycling of different compounds can slow the evolution target site resistance because individuals are unlikely to have target site resistance to all of them. In contrast quantitative resistance can confer resistance across compounds, and so cycling compounds only increases the overall selection pressure, increasing the rate that resistance evolves \cite{Mena2016}. If populations have both target site and quantitative resistance managers may face an impossible situation, needing to simultaneously lower and increase the dose to slow the evolution of resistance. This may be why populations with survival linked traits of both forms (target site and quantitative) are more likely to benefit from evolutionary rescue under harsh conditions (e.g. repeated herbicide application) \cite{Gomu2010}. Thus, it is important to understand how the evolution of target site resistance and quantitative resistance interact, yet this remains largely unexplored, even theoretically (see \cite{Gomu2010, Deba2015, Yeam2015} for exceptions). 

Because quantitative resistance is conferred by mutations in many more genes \cite{petit2010, Busi2013} than target site resistance, most mutations generate variation in quantitative resistance. Thus, quantitative resistance should evolve faster than target site resistance \cite{Dely2010newPhy}. If costly quantitative resistance has already established in a population, any target site resistant individuals may inherit these costs through out-crossing \cite{Yeam2015}. As a result target site resistant individuals may pay the fitness costs of quantitative resistance, even if they do not require its protection, reducing the advantage of target site resistance and slowing its invasion \cite{Chev2008}. 

Limited dispersal might facilitate this effect by creating areas with high levels of target site resistance near to areas with high quantitative resistance. The area with high quantitative resistance could act as a source of genes which impose a demographic cost \cite{Dely2010, Yeam2015}. On the other hand clumping of genotypes (due to dispersal limitation) could allow target site resistance to take hold more quickly, since individuals with rare genotypes will be more likely to cross with other rare mutants if closely related individuals exist close to each other (i.e. sibling-sibling crosses and parent-child crosses) \cite{Some2017}.

We develop a spatially explicit, density dependent population models of the evolution of target site and quantitative resistance in the same population. We apply this model to the economically important weed, \textit{Alopecurus myosuroides} (Huds.). We test how the initial conditions, with respect to population size and the level of quantitative resistance, influence the  evolution of target site resistance. We find that target site and quantitative resistance can co-exist over ecologically and management relevant time scales. Which form of resistance dominates over the short to medium term depends on the local genetic background and the degree of dispersal limitation. 

\section*{Materials and Methods}
% For figure citations, please use "Fig" instead of "Figure".
\subsection*{model}
We model selection on a single locus that confers perfect resistance with no fitness costs (perfect target site resistance) in an annual, monoecious, diploid plant population. The population harbours additive genetic variance for an independent, costly, quantitative resistance trait. We begin with a spatially implicit model and then develop a spatial model where the population exists on a 1D landscape. We use a discrete time, continuous space, modelling framework that can track the evolution of both continuous (quantitative resistance) and discrete (target site resistance) state variables. We assume density dependent seed production, a constant environment, and no demographic stochasticity (i.e. deterministic dynamics).      

Our annual projection interval starts at the beginning of the growing season before any seeds have emerged from the seed bank. Once seeds germinate they are exposed to herbicide and the resistance phenotype is survival.  Quantitative resistance is modelled assuming the infinitesimal model of inheritance \cite{Fish1918}. We assume target site resistance is controlled by a single locus, with Mendelian inheritance, and that the resistant allele is dominant. We assume no mutations in the target site resistance locus. Those individuals that survive then flower and spread pollen. Finally, survivors disperse their seeds into the seed bank. See Fig. \ref{fig:schematic} for a description of the life cycle.  

\begin{figure}[!h]
	\centering
	\includegraphics[width=100mm]{/Users/shauncoutts/Dropbox/projects/MHR_blackgrass/BG_population_model/writting/figures/spatial_model_fig/life_cycle_schem_nospace.pdf}
\caption{\bf Schematic of the spatial population model.} The seed bank has one of three target site resistance genotypes $G \in \{RR, Rr, rr\}$ (where $rr$ indivudals are suceptable) and are distributed over quantitative resistance $g$. The emergence and survival functions are applied to the seed bank distributions to derive the above ground parent distributions. Those parent distributions are mixed to create the new seed distributions. \label{fig:schematic}
\end{figure}

The model projects the seed bank for target site genotype $G \in \{RR, Rr, rr\}$ at time $t$, $b_G(g, t)$, to the seed bank at time $t+1$. $b_G(g, t)$ is a state distribution function of individuals in the seed bank with genotype $G$ and quantitative resistance breeding value $g$. Note that $b_G(g, t)$ is an un-normalized density, and so does not integrate to 1. Because $b_G(g, t)$ is an un-normalized distribution over the breeding value of quantitative resistance, $g$, so are the distributions that arise from $b_G(g, t)$ (i.e. $n_G(g, t)$, $n'_G(g, t)$, $n''_G(g, t)$). When we require a probability density we explicitly normalize. The continuous variable $g$ is inside brackets, while the discrete target site resistance genotype ($G$) is denoted with a subscript and/or a superscript.    

The number of individuals of target site genotype $G$ that emerge from the seed bank and establish at time $t$ is 
\begin{equation}\label{eq:above_ground}
	n_G(g, t) = b_G(g, t)\phi_e,
\end{equation}
where $\phi_e$ is the probability that a seed germinates. The distribution of surviving individuals is 
\begin{equation}\label{eq:abg_sur}
	n'_G(g, t) = n_G(g, t)s_G(g, h) 
\end{equation}
where survival is a function on the target site genotype, $G$, quantitative resistance, $g$, and herbicide application index $h \in \{0, 1\}$.   
\begin{equation}\label{eq:sur_G}
	s_G(g, h) = \begin{cases} 
		\frac{1}{1 + e^{-s_0}} &\text{~if~} G \in \{RR, Rr\} \\
		\frac{(1 - \varsigma)}{1 + e^{-s_0}} + \frac{\varsigma}{1 + e^{-\left(s_0 - h\left(\xi - \textbf{min}(\xi, \rho g) \right)\right)}} &\text{~otherwise~} 		
	\end{cases} 
\end{equation}  
Where $s_0$ determines survival in the absence of herbicide, $\varsigma$ is the proportion of the population that is exposed to herbicide, $\xi$ is the reduction of survival (in logits) due to herbicide and $\rho$ is the resistance induced by a one unit increase in $g$.   

The effects of density and the demographic costs of resistance are applied to the distribution of survivors. These effects could affect survival or reproduction, or both. In an annual plant a reduction in fecundity is equivalent to a reduction in survival, as both affect the number of seeds that enter the seed bank. We define the effective reproductive population ($n''_G(g, t)$) as the population after herbicide application, incorporating the demographic costs of resistance and effects of density. 
\begin{equation}\label{eq:effect_pop}
	n''_G(g, t) = \frac{n'_G(g, t)}{1 + \text{exp}(-(f_c^0 - f_c|g|))}\cdot\frac{1}{1 + f_d\sum_{\forall G} \int_g n'_G(g, t)\text{d}g}
\end{equation} 
where $f_c^0$ defines the fitness cost of quantitative resistance when its breeding value $g = 0$, $|g|$ is the absolute value of $g$ and $f_c$ is the cost of resistance, the logit reduction in effective population size for every one unit increase in $g$. Density dependence is controlled by $f_d$, with $1/f_d$ the population density where individuals start to interfere with each other. 

Our model focuses on a monoecious species, thus $n''_G(g, x, t)$ is both the maternal and paternal parent distributions. We assume that reproduction is not pollen limited. The probability density function of pollen with quantitative resistance $g$ and target site genotype $G$ is 
\begin{equation}\label{eq:pollen_func}
\gamma_G(g, t) = \frac{n''_G(g, t)} {\sum_{\forall G_p}\int_{g_p} n''_{G_p}(g_p, t) \text{d}g_p}, 
\end{equation}

The distribution over $g$ of seeds produced depends on both the paternal and maternal parent distribution. Thus, we must calculate the distribution of seeds over $g$ for each $G_m \times G_p$ cross, 
\begin{equation}
\label{eq:fec_GG}
	\eta_{G_p}^{G_m}(g, t) = \int_{g_m}\int_{g_p} N(g|0.5 g_m + 0.5 g_p, V_a)f_\text{max} n''_{G_m}(g_m, t)\gamma_{G_p}(g_p, t)\text{d}g_p\text{d}g_m
\end{equation}          
$G_p$ and $G_m$ denote the paternal and maternal target site genotypes respectively. Similarly, the maternal and paternal quantitative resistance breeding value are denoted $g_m$ and $g_p$. The offspring produced by every pair of $g_m$:$g_p$ values are assumed to be normally distributed with a mean of $0.5g_m + 0.5g_p$ and variance of $V_a$ (the additive variance) \cite{Ture1994}. $f_\text{max}$ is the maximum possible number of seeds per individual, when both density and quantitative resistance are 0.

To get the distribution of seeds over $g$, for each target site genotype, $G$, $f_{G}(g, t)$, we sum the distributions of seeds from each $G_m \times G_p$ cross, weighting the distributions by the proportion of seeds of genotype $G$ each cross will produce. 
\begin{subequations}
\label{eq:fec_G}
\begin{align}
	f_{RR}(g, t) &= \eta_{RR}^{RR}(g, t) + \eta_{RR}^{Rr}(g, t)0.5 + \eta_{Rr}^{RR}(g, t)0.5 + \eta_{Rr}^{Rr}(g, t)0.25\\
\begin{split}
	f_{Rr}(g, t) &= \eta_{RR}^{Rr}(g, t)0.5 + \eta_{Rr}^{RR}(g, t)0.5 + \eta_{RR}^{rr}(g, t) + \eta_{Rr}^{Rr}(g, t)0.5 + \eta_{Rr}^{rr}(g, t)0.5 +\\
	 &~~~~\eta_{rr}^{RR}(g, t) + \eta_{rr}^{Rr}(g, t)0.5
\end{split}\\
	f_{rr}(g, t) &= \eta_{rr}^{rr}(g, t) + \eta_{rr}^{Rr}(g, t)0.5 + \eta_{Rr}^{rr}(g, t)0.5 + \eta_{Rr}^{Rr}(g, t)0.25
\end{align}  
\end{subequations}  

We close the life cycle by adding the seeds produced during time step $t$, to the seed bank (Eq. \ref{eq:above_ground}) so that 
\begin{equation}
	b_G(g, t + 1) = b_G(g, t)(1 - \phi_e)\phi_b + f_G(g, t).  
\end{equation}
Where $(1 - \phi_e)$ is the proportion of seeds that did not germinate $\phi_b$ is the probability that a seed in the seed bank survives one year. 

To test the effect dispersal limitation had on the evolutionary dynamics we built a 1D spatial version of the model. The full spatial model is set out in \nameref{S1_Appendix}. The main difference is that we introduce a location state variable $x$ so that the seed bank becomes location specific, $b_G(g, x, t)$, and all the density state functions that arise from it ($n_G(g, x, t)$, ect.) also become location specific. Populations and different locations interact via the pollen and seed dispersal functions. The probability density function for pollen arriving at location $x$ (non-spatial version Eq. \ref{eq:pollen_func}) now requires integration over the pollen arriving at location $x$ from all other locations ($x_p$)  
\begin{equation}\label{eq:pollen_func_space}
\gamma_G(g, x, t) = \frac{\int_{x_p} n''_G(g, x, t)d_p(x, x_p)\text{d}x_p} {\sum_{\forall G_p}\int_{x_p}\int_{g_p} n''_{G_p}(g_p, t)d_p(x, x_p) \text{d}g_p\text{d}x_p} 
\end{equation}     
Where the pollen dispersal kernel is 
\begin{equation}\label{eq:pollen_disp}
	d_p(x, x') = \frac{c}{a^{2/c}\Gamma\left(\dfrac{2}{c} \right)\Gamma\left(1 - \dfrac{2}{c} \right)}\left( 1 + \dfrac{\delta_{x,x'}^c}{a} \right)^{-1} 
\end{equation} 
This is a logistic kernel, which was found to be the one of the best fitting pollen dispersal kernels for oil seed rape \cite{Klei2006}. This is a two parameter kernel with a scale, $a$, and shape, $c$, parameter, where $\delta_{x,x'}$ is the distance between locations $x$ and $x'$ and $\Gamma(\cdot)$ is the gamma function. 

The probability that a seed produced at location $x$ is dispersed to location $x'$ is 
\begin{subequations}\label{eq:seed_disp}
\begin{equation}\label{eq:seed_kern}
	d_m(x, x') = \alpha \Upsilon_1 \Omega_1 \delta_{x,x'}^{\Omega_1 - 2} e^{-\Upsilon_1 \delta_{x,x'}^{\Omega_1}} + (1 - \alpha) \Upsilon_2 \Omega_2 \delta_{x,x'}^{\Omega_2 - 2} e^{-\Upsilon_2 \delta_{x,x'}^{\Omega_2}}  
\end{equation}
\begin{equation}\label{eq:shape}
	\Omega_k = \frac{1}{1 + \text{ln}(1 - \omega_k)}
\end{equation}
\begin{equation}\label{eq:scale}
	\Upsilon_k = \frac{\Omega_k - 1}{\Omega_k \mu_k^{\Omega_k}}
\end{equation}
\end{subequations} 
This double Weibull dispersal kernel was found to be the best fit to black grass seed dispersal in a majority of cases \cite{Colb2001}. $\alpha$ is the proportion of seeds in the short dispersal kernel, $\mu_k$ is the distance most seeds disperse to under kernel $k \in \{1, 2\}$. The skew of kernel $k$ is controlled by $\omega_k$, the proportion of seeds that disperse up to distance $\mu_k$. 
 
The model was implemented in Julia version 0.5.0, and code for the model and plotting is available in \nameref{S1_File}.
% Place figure captions after the first paragraph in which they are cited.
%\begin{figure}[!h]
%\caption{{\bf Bold the figure title.}
%Figure caption text here, please use this space for the figure panel descriptions instead of using subfigure commands. A: Lorem ipsum dolor sit amet. B: Consectetur adipiscing elit.}
%\label{fig1}
%\end{figure}

\subsection*{Study system and parametrization}
We parametrize this model using \textit{A. myosuroides} is an annual, diploid, out-crossing grass that is one of the most serious agricultural weeds in Europe \cite{Moss2007}. Due to decades of intensive herbicide use \textit{A. myosuroides} has evolved widespread target site \cite{Moss2007} and quantitative resistance \cite{Yu2014}.   

Several of the population model parameters, particularly those relating to the quantitative genetic selection model, are unknown for our study system. We used field-derived estimates of population density to calibrate the model. While we cannot directly parametrize the model with this data, we could use it to calibrate the model and constrain sets of parameter values to those that predicted population sizes in the range of observed population sizes. We outline this procedure in \nameref{S2_Appendix}. Estimates and sources for each parameter are given in Table \ref{tab:parameters}.          

\begin{table}[!ht]
\begin{adjustwidth}{-2.25in}{0in} % Comment out/remove adjustwidth environment if table fits in text column.
\centering
\caption{
{\bf Model parameters with range used in parameter filtering (see \nameref{S2_Appendix}), etimated value, brief description and source}}
\begin{tabular}{|l+l|l|p{9.5cm}|p{2.5cm}|}
\hline
		{\bf parameter} & {\bf range} & {\bf estimate} & {\bf description} & {\bf source}\\
 \thickhline
 &\multicolumn{4}{l|}{{\it Population model}}\\ \hline
	$\phi_b$ & 0.22 -- 0.79 & 0.42 & seed survival probability & \cite{Thom1997}\\ \hline
	$\phi_e$ & 0.45 -- 0.6 & 0.52 & germination probability & \cite{Colb2006}\\ \hline	
	$f_\text{max}$ & 30 -- 300$^\blacklozenge$ & 45 & seed production (in seeds/plant) of highly susceptible individuals at low densities & \cite{Doyl1986}\\ \hline
	$f_d$ & 0 -- 0.15$^\dag$ & 0.004 & reciprocal of population (1/pop.) at which individuals interfere with each others fecundity & \cite{Doyl1986}\\ \hline 
	$f_c^0$ & 4 -- 10$^\dag$ & no est$^\ddag$  & fecundity in a naive population in logit($f_0$)$f_\text{max}$ & simulation\\ \hline
	$f_c$ & $0.1f_0$ -- $2f_0 ^\dag$ & no est$^\ddag$ & reduction in fecundity (in logits) due to a one unit increase in resistance. Only meaningful in relation to $f_0$ & simulation\\ \hline
	$V_a$ & 1, 2 &  & Variance of the offspring distribution (additive variance) & varied to explore effect\\ \hline
	$s_0$ & fixed & 10 & survival in a naive population is logit($s_0$) & fixed, use $\xi$ and $\rho$ to control survival function\\ \hline
	&\multicolumn{4}{l|}{{\it Management effects}}\\ \hline
	$\varsigma$ & 0.5 -- 1$^\dag$ & 0.8$^\blacklozenge$ & proportion of above ground individuals exposed to herbicide & HGCA\\ \hline   		
	$\xi$ & $2s_0$ -- $3s_0^\dag$ & no est$^\ddag$ & reduction in survival (in logits) due to herbicide (only meaningful in relation to $s_0$) & simulation\\ \hline	
	$\rho$ & $0.1\xi$ -- $2\xi$ & no est$^\ddag$ & protection against herbicide (in logits) conferred by a one unit increase in $g$, only meaningful in relation to $\xi$ & simulation\\ \hline
	int$_{Rr}$ & 0.001 -- 0.2 & & initial frequency of resistant target site genotype. All other individuals are assumed to be of genotype rr & \\ \hline
	&\multicolumn{4}{l|}{{\it Dispersal}}\\ \hline
	$\alpha$ & 0.38 -- 0.58$^\dag$ & 0.48 & proportion of seeds in short dispersal kernel & \cite{Colb2001}\\ \hline   
	$\mu_1$ & 0.46 -- 0.7$^\dag$ & 0.58 & distance (in m) at which maximum number of seeds are found in short seed dispersal kernel & \cite{Colb2001}\\ \hline
	$\mu_2$ & 1.32 -- 1.98$^\dag$ & 1.65 & distance (in m) at which maximum number of seeds are found in long seed dispersal kernel & \cite{Colb2001}\\ \hline
	$\omega_1$ & 0.35 -- 0.53$^\dag$ & 0.44 & proportion of seeds that disperse up to distance $\mu_1$ in short seed dispersal kernel & \cite{Colb2001}\\ \hline
	$\omega_2$ & 0.31 -- 0.47$^\dag$ & 0.39 & proportion of seeds that disperse up to distance $\mu_2$ in long seed dispersal kernel & \cite{Colb2001}\\ \hline
	$a$ & 25.6 --38.4$^\dag$ & 32.3 & scale parameter for pollen dispersal kernel & \cite{Klei2006}\\ \hline
	$c$ & 2.66 -- 3.98$^\dag$ & 3.32 & shape parameter for pollen dispersal kernel & \cite{Klei2006}\\ \hline
\end{tabular}
\begin{flushleft} $\blacklozenge$ sourced from grey literature, unpublished data and expert opinion\\
	$\dag$ range not available from literature, simulation used to find plausible range\\
	$\ddag$ no estimate not available from literature, simulation used to find plausible range
\end{flushleft}
\label{tab:parameters}
\end{adjustwidth}
\end{table}

To asses the behaviour of the model we use \%$R$ to measure the frequency of target site resistance. The level of quantitative resistance in a population was measured as the survival or $rr$ individuals ('survival $rr$').

Unless otherwise stated in these simulation experiments $f_0 = 4$, $s_0 = 10$, $\rho = 1.5$, $f_r = 0.45$ and $\zeta = 16$. With these values an $rr$ individual with quantitative resistance $g = 5$ will have survival of 0.85 under herbicide, but produce $f_{max} 0.85$ seeds and pollen (assuming density is low).

% Results and Discussion can be combined.
\section*{Results and Discussion}
As expected, in the absence of target site resistance, quantitative resistance initially evolved very quickly (within 20 generations). Once quantitative resistance conferred at least 50\% survival we introduced target site resistance at a low frequency (\%R = 0.001). Once introduced target site resistance increased slowly, taking more than 75 generations before target site resistant alleles were 50\% of all target site alleles (solid pink line Fig. \ref{fig:simp_traj}a). In contrast, the control run, where quantitative resistance was forced to remain low (by setting $\rho = 0$), saw target site resistance evolve very quickly when introduced at the same relative frequency (dotted pink line Fig. \ref{fig:simp_traj}a). Quantitative resistance slowed the the evolution of target site resistance so much because it reduced the difference in fitness between target site resistant and target site susceptible individuals.      

\begin{figure}[!h] 
	\includegraphics[height=90mm]{simp_trajectory_HSI.pdf}
\caption{{\bf The evolutionary dynamics of target site (TSR) and quantitative resistance over time under herbicide application (a and b) and the population response (c).} The initial population of 10 seeds had no target site resistance and low quantitative resistance. This population was exposed to herbicide continuously, and target site resistant mutants where introduced (at a frequency of \%R = 0.001) in the first time step after quantitative resistance conferred more than 50\% survival under herbicide. We use three metrics to summarise the evolutionary dynamics, \%R, survival of $rr$ individuals (a) and the fitness advantage to target site resistant individuals (b). The dotted pink line in (a) shows the very rapid evolution of target site resistance under a control run where there was no quantitative resistance (i.e. $\rho = 0$).} 
\label{fig:simp_traj}
\end{figure}

Target site resistant individuals produced nearly twice as many seeds as target site susceptible individuals under herbicide when first introduced (Fig. \ref{fig:simp_traj}b). However, that fitness advantage was quickly reduced to just 0.2 seeds per germinated individual by two processes. First, quantitative resistance continued to evolve after the introduction of target site resistance, increasing target site susceptible survival and thus their average reproduction (since survival is a prerequisite for reproduction). Secondly, fitness costs depend only on the quantitative resistance breeding value ($g$) and not target site resistance. As a result target site resistant individuals can inherit the fitness costs incurred by high quantitative resistance through crossing with target site susceptible individuals, which initially make up the vast majority of the population. These redundant (for target site resistant mutants) fitness costs keep the fitness difference between target site susceptible and resistant mutants low for a prolonged period (50 generations, Fig. \ref{fig:simp_traj}b). 

While the initial evolution of quantitative resistance allowed rapid population growth under herbicide, the subsequent establishment of target site resistance occurred with little impact on the population (Fig. \ref{fig:simp_traj}c and Fig. \ref{fig:evo_pop}b; note the logged x-axis exaggerates population changes that happened after 10 generations). Even though the target site mutants had a large effect on the evolutionary dynamics (Fig \ref{fig:evo_pop}a). 

\begin{figure}[!h] 
	\includegraphics[height=70mm]{evo_pop_plt.pdf}
\caption{{\bf The evolutionary dynamics of target site (TSR) and quantitative resistance over time under herbicide application (a and c) and the population response (b and d) under different initial levels of quantitative resistance and frequency of target site resistance.} The initial population of 10 seeds had no target site resistance and low quantitative resistance. This population was exposed to herbicide continuously, and target site resistant mutants where introduced in the first time step after quantitative resistance conferred more than 50\% (HIGH scenario; a, b) or 1\% (LOW scenario; c, d) survival under herbicide. In plots of resistance trait space (a, c) points mark 2 generation intervals and numbers in the markers show 20 generation intervals, thus the closer markers are together the slower the evolutionary dynamics.} 
\label{fig:evo_pop}
\end{figure}

As expected, the rate that target site resistance could evolve was very dependent on the genetic background into which the target site resistance was introduced [REF]. When quantitative resistance was low target site resistance evolved rapidly, reaching \%R $>$ 0.7 within 10 generations, unless target site resistant alleles ($R$) were introduced at a low frequency, and even then \%R $>$ 0.7 was reached within 20 generations (Fig. \ref{fig:evo_pop}c).

When high quantitative resistance had already developed two different dynamics occur depending on the initial frequency of target site resistance (Fig. \ref{fig:evo_pop}a). When initial \%R = 0.1, target site resistance could develop rapidly, with target site resistant alleles making 50\% of all target site alleles within 10 generations (dark green, Fig. \ref{fig:evo_pop}a). At the same time quantitative resistance initially increases slightly, but then the dynamic seen in Fig. \ref{fig:simp_traj}b starts to drive down the level of quantitative resistance within four generations. 

In contrast, when target site resistance was introduced at a lower frequency (\%R $\leq$ 0.01) quantitative resistance continued to increase rapidly for the next 10 generations, and did not begin to decline until after 20 generations. In these cases the decline in quantitative resistance started before substantial levels of target site resistance had built up in the population (\%R $<$ 0.1; Fig \ref{fig:evo_pop}a). To explore this dynamic in more detail we focus on the scenario where target site resistant alleles were introduced to a population that already had high quantitative resistance at a frequency of \%R = 0.001 in Fig. \ref{fig:G_fit}. 

\begin{figure}[!h] 
	\includegraphics[height=70mm]{fit_fec_plt.pdf}
\caption{{\bf The average reproductive performance (as a measure of fitness costs) of the three target site genotypes, $RR$, $Rr$, $rr$, as target site resistance (measured by \%R) evolves over time.} The initial population of 10 seeds had no target site resistance and low quantitative resistance. This population was exposed to herbicide continuously, and target site resistant mutants where introduced (at a frequency of \%R = 0.001) in the first time step after quantitative resistance conferred more than 50\% survival under herbicide. Points mark 2 generation intervals and numbers in the markers show 20 generation intervals. The closer markers are together the slower the evolutionary dynamics.} 
\label{fig:G_fit}
\end{figure}

The average reproductive performance of each target site genotype shows the fitness costs incurred from quantitative resistance (Fig. \ref{fig:G_fit}). Initially the fitness costs of the target site resistant portion of the population increase rapidly as the fitness costs of the target site susceptible part of the population is crossed into the target site resistant part. This reduced the selection pressure for target site resistance and slowed its evolution (step initial drop in darker lines Fig. \ref{fig:G_fit}). The fitness (and by extension the breeding value of quantitative resistance $g$) moved in unison between the three target site genotypes because all three genotypes mixed quantitatively, although in the case of $RR$ and $rr$ this mixing occurred via an intermediate cross with $Rr$. A lower breeding value benefited target site resistant parents (since their survival is not dependent on $g$), but not the target site susceptible parents. The small fitness advantage for target site parents with a lower breeding value ($g$) selects for lower $g$ in that part of the population. This lower $g$ value is then mixed back into the target site susceptible part of the population. Although this mixing happens very slowly at first because target site resistance is rare, it starts off a weak feed back. As the lower breeding value mixes through the whole population it only decreases the survival of the target site susceptible part of the population, increasing the fitness advantage of target site resistant parents, and so increasing the rate of evolution of target site resistance.                           

As in the non-spatial model, in the spatial model target site resistance evolved rapidly at the introduction location when the resident population had low levels of quantitative resistance, and evolved much more slowly when the resident population had high levels of quantitative resistance (Fig. \ref{fig_spread}). However, in both cases the spread of target site resistance across the landscape was much slower than its initial evolution at the introduction location. This lead to very steep clines in target site resistance. As an example, in Fig. \ref{fig_spread}c 10 generations after target site resistant alleles were introduced to the landscape they made up nearly 70\% of all target site alleles at the introduction site. It took 30 generations to reach a similar level of target site resistance at a location just 25m away, this is despite the pollen dispersal kernel being flat at this scale. 

\begin{figure}[!h] 
\includegraphics[height=80mm]{space_time_rr_thresh.pdf} 
\caption{{\bf The spread of target site resistance over a 1D landscape that already contains a target site susceptible population.} The resident population was exposed to herbicide continuously, and target site resistant mutants where introduced (at a frequency of \%R = 0.1) only to the center location in the landscape, in the first time step after quantitative resistance conferred more than 50\% (HIGH scenario; a, b) or 1\% (LOW scenario; c, d) survival under herbicide. Each line shows \%R at 10 generation intervals for each location.} 
\label{fig_spread}
\end{figure}

This occurred because dispersal limitation meant that at the introduction location target site resistant parents were more likely to cross with each other, and their offspring were dispersed to nearby locations (and thus more likely to cross with each other). However, dispersal limitation also meant that target site resistance alleles arriving at a location away from the introduction location were swamped by the much higher frequency of locally produced target site susceptible alleles.

The initial sate of the population also changed the direction of the effect of additive variance ($V_a$) on the evolution of target site resistance. When the resident population had low levels of quantitative resistance higher $V_a$ resulted in faster evolution of target site resistance (Fig. \ref{fig_spread}c,d). In this scenario the target site resistant genotype initially had an enormous fitness advantage (germinated target site resistant parents had fecundity more than 20 times higher than target site susceptible parents, mainly due to differences in survival). This allowed target site resistance to evolve very quickly at the introduction site (\%R $>$ 0.65 after 10 generations Fig. \ref{fig_spread}c). This reduced the evolution of quantitative resistance, as the target site resistant part of the population had a lower breeding value ($g$), which was mixed across the whole population (Fig. \ref{fig:G_fit}). This set up a race, the faster quantitative resistance could evolve from a low level (which occurred with higher values of $V_a$), the faster the fitness advantage of target site resistance was reduced. This slowed the initial (and thus also the subsequent) evolution of target site resistance across the landscape.             

In contrast when the resident population had already evolved high levels of quantitative resistance before target site resistance was introduced higher values of $V_a$ lead to faster evolution and spread of target site resistance (Fig. \ref{fig_spread}). As in the non-spatial model the fitness advantage quickly dropped (Fig. \ref{fig:ts_TSR_adv}) as the target site resistant immigrants crossed with the resident population and inherited the fitness costs of high quantitative resistance. The fitness advantage only started to increase once the the population began to shed the fitness costs of quantitative resistance by evolving a lower breeding value, $g$ (via the same process as the non-spatial model, Fig. \ref{fig:G_fit}). This process was much faster when $V_a$ was higher (Fig. \ref{fig:ts_TSR_adv} dark vs light lines) because there was more variation in quantitative resistance to act on when $V_a$ was higher. Having a higher frequency of target site resistance also helped the population shed the fitness costs as it reduced the influence of the target site susceptible part of the population (which has a higher breeding value) on the breeding value of the whole population (recall the breeding value of all target site genotypes track each other; Fig. \ref{fig:G_fit}). 

\begin{figure}[!h] 
\includegraphics[height=60mm]{space_time_TSR_adv.pdf} 
\caption{{\bf The fitness advantage of target site resistant plants as target site resistance evolves over time at the location target site resistance was introduced (a) and 30m from the introduction location (b).} The resident population was exposed to herbicide continuously, and target site resistant mutants where introduced (at a frequency of \%R = 0.1) only to the center location in the landscape, in the first time step after quantitative resistance conferred more than 50\% survival under herbicide (HIGH scenario). Points mark 2 generation intervals and numbers in the markers show 20 generation intervals. Thus the closer markers are together the slower the evolutionary dynamics.} 
\label{fig:ts_TSR_adv}
\end{figure}

A lower breeding value, $g$, both decreased survival for the target site susceptible genotype, and increased fecundity for the target site resistant genotype, increasing the fitness advantage of target site resistance, further increasing the frequency of target site resistance. This positive feedback is why the increase in fitness advantage accelerates as \%R increased (curve upwards in Fig. \ref{fig:ts_TSR_adv}). This dynamic was much slower just 30m from the introduction site since the initial frequency of target site resistance alleles was lower in that part of the landscape, greatly slowing the initial evolution of target site resistance (Fig. \ref{fig:ts_TSR_adv}b).     

\section*{Discussion}
We find that the evolution of target site resistance was highly dependent on the genetic background of the population it was introduced into, with respect to quantitative resistance. Whether the population had high or low levels of quantitative resistance reversed the effect of additive variance in  quantitative resistance and the effect of dispersal limitation on the evolution of target site resistance. We also show a large parameter space where both target site and quantitative resistance co-existed over the medium term, relevant to management in agricultural systems. This occurred even though target site resistant individuals had a substantial fitness advantage. Further, the co-existence (or not) of target site and quantitative resistance occurred in different parts of the parameter space depending on genetic background of the existing population. Given this high level of variability and strong context dependence we expect to see target site or quantitative resistance (or both) predominate in different regions of a landscape. To date there is little evidence to either support or refute this hypothesis because the prevalence and type of resistance has not typically been studied at landscape scales \cite{Dely2013}.

We show the frequency of target site resistance changes rapidly over space, and invade very slowly into existing populations. In previous models it has been suggested that this slow spread is due to limited dispersal of seeds and pollen \cite{Some2017}. However, in our case the landscape is small relative to the pollen dispersal kernel in particular, and the spread of target site resistance would have been faster had it just been limited by dispersal. Instead, our slow spread for target site resistance was due to the local genetic background. Our pollen dispersal kernel had a flat tail at the scale of the modelled landscape we modelled, which meant that while some pollen could disperse over the whole landscape, the majority of pollen went to location near the parent plant, and any single location away from a target site resistant father received only a small amount of target site resistant pollen. Being rare, this pollen crossed with target site susceptible mothers, producing only a few heterozygous target site resistant offspring. Further, these offspring inherited many of the demographic costs of quantitative resistance, reducing their fitness to be closer to their target site susceptible neighbours \cite{Chev2008}. In the next generation these offspring predominantly crossed with target site susceptible individuals, reinforcing this process. We expect this effect of the genetic background to be even stronger in 2D. On the 1D landscape the genetic background of the neighbours could only effect a location from two directions. On a 2D landscape every location has many more neighbours, and thus is more affected by their genetic background.	                

These spatial dynamics suggests that applying herbicide in a spatial mosaic (a strategy recommended to slow the evolution of resistance \cite{Rex2013}), may be effective in managing target site resistance, especially when target site resistance is spreading into an existing population. We also found co-existence of both target site and quantitative resistance across the whole population when the invasion occurred into an empty landscape and the source population (introduced seeds) had high levels of quantitative resistance. And this co-existence could persist for more than 100 generations. This suggests co-existence will be most common at the leading edge of an invasion, where expanding nascent population foci are common \cite{Mooy1988}. As a result, it is almost inevitable that both target site and quantitative resistance will co-exist somewhere in the landscape.

This has important implications for the management of resistance. The continuous nature of quantitative resistance means that there is always some variation for selection to act on and so it can develop earlier \cite{Dely2010newPhy}. Quantitative resistance can also give cross resistance between compounds \cite{Bauc2016, Neve2007}. While target site resistance can confer high levels of resistance for little demographic cost \cite{Bauc2016}. Places where both forms of resistance co-exist may experience the worst of both worlds, resistance that develops very quickly, which is replaced by target site resistance that confers very high levels of resistance \cite{Vera2015}. These differences in behaviour mean different strategies are used to manage target site and quantitative resistance (for example compound cycling to delay the evolution of target site resistance \cite{Rex2013}) \cite{Gard1998}. When populations have both target site and quantitative resistance we may find ourselves with few good strategies to slow the evolution of resistance \cite{Gard1998}. 

\section*{Conclusion}
Our results suggest that the genetic background against which target site resistance evolves has an enormous effect on how fast and under which conditions target site or quantitative resistance dominates over the short and medium term. As a result we should expect the genetic basis of resistance to be heterogeneous at both local and landscape scales. It follows from this that the effectiveness of different resistance management strategies, like compound cycling and very high doses, may also vary over space, as these different strategies exploit different weaknesses target site and quantitative resistance \cite{Gard1998, Rex2013}. Our model is a first step in generating hypothesis on how target site and quantitative resistance interact interact across space. It is imperative that this work is followed by empirical observations and experimental tests. A logical starting point would be spatially extensive surveys of resistant populations to see where and when target site and quantitative resistance dominate or co-exist.

\section*{Supporting Information}

% Include only the SI item label in the paragraph heading. Use the \nameref{label} command to cite SI items in the text.
\paragraph*{S1 File.}
\label{S1_File}
{\bf Julia implementation of model and plotting code}  

\paragraph*{S1 Appendix.}
\label{S1_Appendix}
{\bf Spatial model.} 

\paragraph*{S2 Appendix.}
\label{S2_Appendix}
{\bf Parameter calibration} 

\section*{Acknowledgements}
We thank some people for some things

\nolinenumbers

% Either type in your references using
% \begin{thebibliography}{}
% \bibitem{}
% Text
% \end{thebibliography}
%
% or
% % Compile your BiBTeX database using our plos2015.bst
% style file and paste the contents of your .bbl file
% here.
% 
%\begin{thebibliography}{10}
%
%\bibitem{bib1}

%Conant GC, Wolfe KH.
%\newblock {{T}urning a hobby into a job: how duplicated genes find new
%  functions}.
%\newblock Nat Rev Genet. 2008 Dec;9(12):938--950.

%\end{thebibliography}

%REMEMBER TO FIX UP THESE CITATIONS AND USE THEIR FORMATTING ONCE I HAVE A MORE COMPLEATE VERSION.

%\bibliographystyle{/home/shauncoutts/Dropbox/shauns_paper/referencing/bes} 
\bibliography{/home/shauncoutts/Dropbox/shauns_paper/referencing/refs}


\end{document}

