% Template for PLoS
% Version 3.3 June 2016
%
% % % % % % % % % % % % % % % % % % % % % %
%
% -- IMPORTANT NOTE
%
% This template contains comments intended 
% to minimize problems and delays during our production 
% process. Please follow the template instructions
% whenever possible.
%
% % % % % % % % % % % % % % % % % % % % % % % 
%
% Once your paper is accepted for publication, 
% PLEASE REMOVE ALL TRACKED CHANGES in this file 
% and leave only the final text of your manuscript. 
% PLOS recommends the use of latexdiff to track changes during review, as this will help to maintain a clean tex file.
% Visit https://www.ctan.org/pkg/latexdiff?lang=en for info or contact us at latex@plos.org.
%
%
% There are no restrictions on package use within the LaTeX files except that 
% no packages listed in the template may be deleted.
%
% Please do not include colors or graphics in the text.
%
% The manuscript LaTeX source should be contained within a single file (do not use \input, \externaldocument, or similar commands).
%
% % % % % % % % % % % % % % % % % % % % % % %
%
% -- FIGURES AND TABLES
%
% Please include tables/figure captions directly after the paragraph where they are first cited in the text.
%
% DO NOT INCLUDE GRAPHICS IN YOUR MANUSCRIPT
% - Figures should be uploaded separately from your manuscript file. 
% - Figures generated using LaTeX should be extracted and removed from the PDF before submission. 
% - Figures containing multiple panels/subfigures must be combined into one image file before submission.
% For figure citations, please use "Fig" instead of "Figure".
% See http://journals.plos.org/plosone/s/figures for PLOS figure guidelines.
%
% Tables should be cell-based and may not contain:
% - spacing/line breaks within cells to alter layout or alignment
% - do not nest tabular environments (no tabular environments within tabular environments)
% - no graphics or colored text (cell background color/shading OK)
% See http://journals.plos.org/plosone/s/tables for table guidelines.
%
% For tables that exceed the width of the text column, use the adjustwidth environment as illustrated in the example table in text below.
%
% % % % % % % % % % % % % % % % % % % % % % % %
%
% -- EQUATIONS, MATH SYMBOLS, SUBSCRIPTS, AND SUPERSCRIPTS
%
% IMPORTANT
% Below are a few tips to help format your equations and other special characters according to our specifications. For more tips to help reduce the possibility of formatting errors during conversion, please see our LaTeX guidelines at http://journals.plos.org/plosone/s/latex
%
% For inline equations, please be sure to include all portions of an equation in the math environment.  For example, x$^2$ is incorrect; this should be formatted as $x^2$ (or $\mathrm{x}^2$ if the romanized font is desired).
%
% Do not include text that is not math in the math environment. For example, CO2 should be written as CO\textsubscript{2} instead of CO$_2$.
%
% Please add line breaks to long display equations when possible in order to fit size of the column. 
%
% For inline equations, please do not include punctuation (commas, etc) within the math environment unless this is part of the equation.
%
% When adding superscript or subscripts outside of brackets/braces, please group using {}.  For example, change "[U(D,E,\gamma)]^2" to "{[U(D,E,\gamma)]}^2". 
%
% Do not use \cal for caligraphic font.  Instead, use \mathcal{}
%
% % % % % % % % % % % % % % % % % % % % % % % % 
%
% Please contact latex@plos.org with any questions.
%
% % % % % % % % % % % % % % % % % % % % % % % %

\documentclass[10pt,letterpaper]{article}
\usepackage[top=0.85in,left=2.75in,footskip=0.75in]{geometry}

% amsmath and amssymb packages, useful for mathematical formulas and symbols
\usepackage{amsmath,amssymb}

% Use adjustwidth environment to exceed column width (see example table in text)
\usepackage{changepage}

% Use Unicode characters when possible
\usepackage[utf8x]{inputenc}

% textcomp package and marvosym package for additional characters
\usepackage{textcomp,marvosym}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

% makes the \textsubscript work on my older version of latex
\usepackage{fixltx2e}

% Use nameref to cite supporting information files (see Supporting Information section for more info)
\usepackage{nameref,hyperref}

% line numbers
\usepackage[right]{lineno}

% ligatures disabled
\usepackage{microtype}
\DisableLigatures[f]{encoding = *, family = * }

% color can be used to apply background shading to table cells only
\usepackage[table]{xcolor}
\usepackage{multirow} %connecting columns in tables
\usepackage{multicol}
\usepackage{longtable}

% array package and thick rules for tables
\usepackage{array}

% create "+" rule type for thick vertical lines
\newcolumntype{+}{!{\vrule width 2pt}}

% create \thickcline for thick horizontal lines of variable length
\newlength\savedwidth
\newcommand\thickcline[1]{%
  \noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
  \cline{#1}%
  \noalign{\vskip\arrayrulewidth}%
  \noalign{\global\arrayrulewidth\savedwidth}%
}

% \thickhline command for thick horizontal lines that span the table
\newcommand\thickhline{\noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
\hline
\noalign{\global\arrayrulewidth\savedwidth}}


% Remove comment for double spacing
%\usepackage{setspace} 
%\doublespacing

% Text layout
\raggedright
\setlength{\parindent}{0.5cm}
\textwidth 5.25in 
\textheight 8.75in

% Bold the 'Figure #' in the caption and separate it from the title/caption with a period
% Captions will be left justified
\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}
\renewcommand{\figurename}{Fig}

% Use the PLoS provided BiBTeX style
\bibliographystyle{plos2015}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother

% Leave date blank
\date{}

% Header and Footer with logo
\usepackage{lastpage,fancyhdr,graphicx}
\usepackage{epstopdf}
\pagestyle{myheadings}
\pagestyle{fancy}
\fancyhf{}
\setlength{\headheight}{27.023pt}
\lhead{\includegraphics[width=2.0in]{PLOS-submission.eps}}
\rfoot{\thepage/\pageref{LastPage}}
\renewcommand{\footrule}{\hrule height 2pt \vspace{2mm}}
\fancyheadoffset[L]{2.25in}
\fancyfootoffset[L]{2.25in}
\lfoot{\sf PLOS}

%% Include all macros below

\newcommand{\lorem}{{\bf LOREM}}
\newcommand{\ipsum}{{\bf IPSUM}}

%% END MACROS SECTION


\begin{document}
\vspace*{0.2in}

% Title must be 250 characters or less.
\begin{flushleft}
{\Large
\textbf\newline{Eco-evolutionary Dynamics on the Invasion Front Drives the Co-existence of Target Site and Quantitative Resistance} % Please use "title case" (capitalize all terms in the title except conjunctions, prepositions, and articles).
}
\newline
% Insert author names, affiliations and corresponding author email (do not include titles, positions, or degrees).
\\
Shaun Coutts\textsuperscript{1*},
Rob Freckelton\textsuperscript{1},
Helen Hicks\textsuperscript{1},
Paul Neve\textsuperscript{2},
Dylan Childs\textsuperscript{1},
\\
\bigskip
\textbf{1} Animal and Plant Sciences, University of Sheffield, Sheffield, UK
\\
\textbf{2} Affiliation Dept/Program/Center, Institution Name, City, State, Country
\\
\bigskip

% Insert additional author notes using the symbols described below. Insert symbol callouts after author names as necessary.
% 
% Remove or comment out the author notes below if they aren't used.
%
% Primary Equal Contribution Note
%\Yinyang These authors contributed equally to this work.

% Additional Equal Contribution Note
% Also use this double-dagger symbol for special authorship notes, such as senior authorship.
%\ddag These authors also contributed equally to this work.

% Current address notes
%\textcurrency Current Address: Dept/Program/Center, Institution Name, City, State, Country % change symbol to "\textcurrency a" if more than one current address note
% \textcurrency b Insert second current address 
% \textcurrency c Insert third current address

% Deceased author note
%\dag Deceased

% Group/Consortium Author Note
%\textpilcrow Membership list can be found in the Acknowledgments section.

% Use the asterisk to denote corresponding authorship and provide email address in note below.
* shaun.coutts@gmail.com

\end{flushleft}
% Please keep the abstract below 300 words
\section*{Abstract}
Evolved resistance threatens the use of important antibiotics, pesticides and herbicides (collectively xenobiotics). Xenobiotic resistance is a complex trait, controlled by a variable number of genes. At one extreme target site resistance is conferred by a single mutation, and at the other extreme quantitative resistance is conferred by the small additive effects of many mutations. There is growing evidence that target site and quantitative resistance exist within the same population. This has important implications for understanding how traits evolve, since target site resistance should displace the less effective more costly quantitative resistance. There are also important implications for the management of resistance as strategies that slow the evolution of target site, such as cycling compounds, may increase selective pressure for quantitative resistance. We model resistance with both target site and quantitative resistance acting in the same population. We apply this model to the economically important weed, \textit{Alopecurus myosuroides} (Huds.). The ecological and historical context determined the which form of resistance evolved. When target site resistance was introduced to an empty landscape it quickly reached fixation in the population. When target site resistance was introduced into a population with prior selection for quantitative resistance its evolution was slower. We also show that spatial structure in both target site and quantitative resistance can either promote or reduce the evolution of target site resistance, depending on the context. Because of this context dependence the full range of outcomes were seen under ecologically realistic scenarios and management relevant time scales; from target site resistance approaching fixation, to all resistance being quantitative, to both target site and quantitative resistance co-existing in the population. Given this strong context dependence we should not be surprised to see both target site and quantitative resistance existing in the same population.     

% Please keep the Author Summary between 150 and 200 words
\section*{Author Summary}
Growing resistance to antibiotics, pesticides and herbicides threaten food security and human health worldwide. Herbicide resistance alone threatens our ability to control weeds, which left unchecked can reduce yields of vital crops by 30\%. Further, resistance can evolve within three years, which means developing new compounds is not an effective solution on its own, we must also prolong the useful life of existing compounds. Complicating this effort is the fact that resistance can be conferred by a highly variable number of mutations, each providing very different levels of resistance. Strategies to control resistance conferred by a single, highly effective mutation (known as target site resistance), such as cycling compounds, can speed up the evolution of resistance conferred by many mutations, each with small effect (known as quantitative resistance), by increasing selection pressure. We model the evolution of both target site and quantitative resistance in the same population and show that the ecological and historical context under which resistance evolves can lead to mainly target site resistance, or high levels of quantitative resistance, or both forms co-existing. This could lead to the worst of both worlds in terms of managing resistance, leaving few viable strategies to slow the evolution of resistance.      

\linenumbers

% Use "Eq" instead of "Equation" for equation citations.
\section*{Introduction}
There is currently a global resistance crisis \cite{Serv2013, Ross2014} and important antibiotics, pesticides and herbicides (collectively xenobiotics) are losing their efficacy \cite{Palu2001}. Together, these chemical tools are a crucial part of the world's food productions system \cite{Duke2012}, and are used worldwide to control life threatening infectious diseases. However, evolving resistance threatens their continued use \cite{Barb2011, Nkya2013}. Economic and regulatory conditions mean that bringing new, safe and effective compounds to market is time consuming and expensive \cite{Duke2012}. In addition, the useful life of new xenobiotics can be as short as three years if their use is not well managed \cite{Palu2001, Duke2012}. To address this crisis and reduce its impact we need to understand the factors that promote or constrain the evolution of resistance. 

Xenobiotic resistance is a complex trait, with a variable number of contributing loci \cite{Warw1991, Neve2007, Dely2013, Bauc2016}, complicating efforts to understand and manage its evolution. The best understood form of resistance is target site resistance \cite{Warw1991, Neve2007, Dely2013, Serv2013}, which is often controlled by mutation at a single locus \cite{Bauc2016}. This mutation alters the binding site of the xenobiotic so that it can no longer bind \cite{Dely2013, Bauc2016}. Target site resistance often confers very high levels of resistance, and incurs few fitness costs (i.e. reductions in survival, growth and/or fecundity)\cite{Warw1994, Vila2005, Bauc2016}. At the other extreme, quantitative resistance is conferred by the small effects of many genes \cite{Land1989, Mack2009, Dely2013, Rajo2013}. Quantitative resistance is usually achieved by metabolizing the xenobiotic, or transporting it away from its binding site \cite{Dely2013, Bauc2016} and often confers lower levels of resistance and incurs fitness costs \cite{Vila2005, Bauc2016}. There is also growing evidence that target site and quantitative resistance exist within the same population in both plants \cite{Warw1991, Vila2005, Herr2014, Han2016} and insects \cite{Gard1998, Donn2009, Bing2011, Hend2013, Oake2013}.

Whether resistance is conferred by target site or quantitative resistance has important implications for its management. For example using lower doses delays the evolution of target site resistance once present in a population, but may accelerate the evolution of quantitative resistance \cite{Gard1998}.
In addition the cycling of different compounds can slow the evolution target site resistance because individuals are unlikely to have target site resistance to all of them. In contrast quantitative resistance can confer resistance across compounds, and so cycling compounds only increases the overall selection pressure, increasing the rate that resistance evolves \cite{Mena2016}. If populations have both target site and quantitative resistance managers may face an impossible situation, needing to simultaneously lower and increase the dose to slow the evolution of resistance. This may be why populations with survival linked traits of both forms (target site and quantitative) are more likely to benefit from evolutionary rescue under harsh conditions (e.g. repeated herbicide application) \cite{Gomu2010}. Thus, it is important to understand how the evolution of target site resistance and quantitative resistance interact, yet this remains largely unexplored, even theoretically (see \cite{Gomu2010, Deba2015, Yeam2015} for exceptions). 

Because quantitative resistance is conferred by mutations in many more genes \cite{petit2010, Busi2013} than and target site resistance most mutations generate variation in quantitative resistance. Thus, quantitative resistance should evolve faster than target site resistance \cite{Dely2010newPhy}. If costly quantitative resistance has already established in a population, any target site resistant individuals may inherit these costs through out-crossing \cite{Yeam2015}. As a result target site resistant individuals may pay the fitness costs of quantitative resistance, even if they do not require its protection, reducing the advantage of target site resistance and slowing its invasion \cite{Chev2008}. 

Limited dispersal might facilitate this effect by creating areas with high levels of target site resistance near to areas with high quantitative resistance. The area with high quantitative resistance could act as a source of genes which impose a demographic cost \cite{Dely2010, Yeam2015}. On the other hand clumping of genotypes (due to dispersal limitation) could allow target site resistance to take hold more quickly, since individuals with rare genotypes will be more likely to cross with other rare mutants if closely related individuals exist close to each other (i.e. sibling-sibling crosses and parent-child crosses).

We develop a spatially explicit, density dependent population models of the evolution of target site and quantitative resistance in the same population. We apply this model to the economically important weed, \textit{Alopecurus myosuroides} (Huds.). We test how the initial conditions, with respect to population size and the level of quantitative resistance, influence the  evolution of target site resistance. We find that target site and quantitative resistance can co-exist over ecologically and management relevant time scales. Which form of resistance dominates over the short to medium term depends on the local genetic background and the degree of dispersal limitation. 

\section*{Materials and Methods}
% For figure citations, please use "Fig" instead of "Figure".
\subsection*{model}
We model selection on a single locus that confers perfect resistance with no fitness costs (perfect target site resistance) in an annual, monoecious, diploid plant population. The population harbours additive genetic variance for an independent, costly, quantitative resistance trait. We begin with a spatially implicit model and then develop spatial model where the population exists on a 1D landscape. We use a discrete time, continuous space, modelling framework that can track the evolution of both continuous (quantitative resistance) and discrete (target site resistance) state variables. We assume density dependent seed production, a constant environment, and no demographic stochasticity (i.e. deterministic dynamics).      

Our annual projection interval starts at the beginning of the growing season before any seeds have emerged from the seed bank. Once seeds germinate they are exposed to herbicide and the resistance phenotype is survival.  Quantitative resistance is modelled assuming the infinitesimal model of inheritance \cite{Fish1918}. We assume target site resistance is controlled by a single locus, with Mendelian inheritance. We assume no mutations in the target site resistance locus. Those individuals that survive then flower and spread pollen. Finally, survivors disperse their seeds into the seed bank. See Fig. \ref{fig:schematic} for a description of the life cycle.  

\begin{figure}[!h]
	\centering
	\includegraphics[width=100mm]{/home/shauncoutts/Dropbox/projects/MHR_blackgrass/BG_population_model/writting/figures/spatial_model_fig/life_cycle_schem_nospace.pdf}
\caption{\bf Schematic of the spatial population model.} The seed bank has one of three target site resistance genotypes $G \in \{RR, Rr, rr\}$ (where $rr$ indivudals are suceptable) and are distributed over quantitative resistance $g$. The emergence and survival functions are applied to the seed bank distributions to derive the above ground parent distributions. Those parent distributions are mixed to create the new seed distributions. \label{fig:schematic}
\end{figure}

The model projects the seed bank for target site genotype $G \in \{RR, Rr, rr\}$ at time $t$, $b_G(g, t)$, to the seed bank at time $t+1$. $b_G(g, t)$ is a state distribution function of individuals in the seed bank with genotype $G$ and quantitative resistance breeding value $g$. Note that $b_G(g, t)$ is an un-normalized density, and so does not integrate to 1. Because $b_G(g, t)$ is an un-normalized distribution over the breeding value of quantitative resistance, $g$, so are the distributions that arise from $b_G(g, t)$ (i.e. $n_G(g, t)$, $n'_G(g, t)$, $n''_G(g, t)$). When we require a probability density we explicitly normalize. The continuous variable $g$ is inside brackets, while the discrete target site resistance genotype ($G$) is denoted with a subscript and/or a superscript.    

The number of individuals of target site genotype $G$ that emerge from the seed bank and establish at time $t$ is 
\begin{equation}\label{eq:above_ground}
	n_G(g, t) = b_G(g, t)\phi_e,
\end{equation}
where $\phi_e$ is the probability that a seed germinates. The distribution of surviving individuals is 
\begin{equation}\label{eq:abg_sur}
	n'_G(g, t) = n_G(g, t)s_G(g, h) 
\end{equation}
where survival is a function on the target site genotype, $G$, quantitative resistance, $g$, and herbicide application index $h \in \{0, 1\}$.   
\begin{equation}\label{eq:sur_G}
	s_G(g, h) = \begin{cases} 
		\frac{1}{1 + e^{-s_0}} &\text{~if~} G \in \{RR, Rr\} \\
		\frac{(1 - \varsigma)}{1 + e^{-s_0}} + \frac{\varsigma}{1 + e^{-\left(s_0 - h\left(\xi - \textbf{min}(\xi, \rho g) \right)\right)}} &\text{~otherwise~} 		
	\end{cases} 
\end{equation}  
Where $s_0$ determines survival in the absence of herbicide, $\varsigma$ is the proportion of the population that is exposed to herbicide, $\xi$ is the reduction of survival (in logits) due to herbicide and $\rho$ is the resistance induced by a one unit increase in $g$.   

The effects of density and the demographic costs of resistance are applied to the distribution of survivors. These effects could affect survival or reproduction, or both. In an annual plant a reduction in fecundity is equivalent to a reduction in survival, as both affect the number of seeds that enter the seed bank. We define the effective reproductive population ($n''_G(g, t)$) as the population after herbicide application, incorporating the demographic costs of resistance and effects of density. 
\begin{equation}\label{eq:effect_pop}
	n''_G(g, t) = \frac{n'_G(g, t)}{1 + \text{exp}(-(f_c^0 - f_c|g|))}\cdot\frac{1}{1 + f_d\sum_{\forall G} \int_g n'_G(g, t)\text{d}g}
\end{equation} 
where $f_c^0$ defines the cost of demographic cost of quantitative resistance when its breeding value $g = 0$, $|g|$ is the absolute value of $g$ and $f_c$ is the cost of resistance, the logit reduction in effective population size for every one unit increase in $g$. Density dependence is controlled by $f_d$, with $1/f_d$ the population density where individuals start to interfere with each other. 

Our model focuses on a monoecious species, thus $n''_G(g, x, t)$ is both the maternal and paternal parent distributions. We assume that reproduction is not pollen limited. The probability density function of pollen with quantitative resistance $g$ and target site genotype $G$ is 
\begin{equation}\label{eq:pollen_func}
\gamma_G(g, t) = \frac{n''_G(g, t)} {\sum_{\forall G_p}\int_{g_p} n''_{G_p}(g_p, t) \text{d}g_p}, 
\end{equation}

The distribution over $g$ of seeds produced depends on both the paternal and maternal parent distribution. Thus, we must calculate the distribution of seeds over $g$ for each $G_m \times G_p$ cross, 
\begin{equation}
\label{eq:fec_GG}
	\eta_{G_p}^{G_m}(g, t) = \int_{g_m}\int_{g_p} N(g|0.5 g_m + 0.5 g_p, V_a)f_\text{max} n''_{G_m}(g_m, t)\gamma_{G_p}(g_p, t)\text{d}g_p\text{d}g_m
\end{equation}          
$G_p$ and $G_m$ denote the paternal and maternal target site genotypes respectively. Similarly, the maternal and paternal quantitative resistance breeding value are denoted $g_m$ and $g_p$. The offspring produced by every pair of $g_m$:$g_p$ values are assumed to be normally distributed with a mean of $0.5g_m + 0.5g_p$ and variance of $V_a$ (the additive variance) \cite{Ture1994}. $f_\text{max}$ is the maximum possible number of seeds per individual, when both density and quantitative resistance are 0.

To get the distribution of seeds over $g$, at each location, for each target site genotype, $G$, $f_{G}(g, t)$, we must sum the distributions of seeds from each $G_m \times G_p$ cross, weighting the distributions by the proportion of seeds of genotype $G$ each cross will produce. 
\begin{subequations}
\label{eq:fec_G}
\begin{align}
	f_{RR}(g, t) &= \eta_{RR}^{RR}(g, t) + \eta_{RR}^{Rr}(g, t)0.5 + \eta_{Rr}^{RR}(g, t)0.5 + \eta_{Rr}^{Rr}(g, t)0.25\\
\begin{split}
	f_{Rr}(g, t) &= \eta_{RR}^{Rr}(g, t)0.5 + \eta_{Rr}^{RR}(g, t)0.5 + \eta_{RR}^{rr}(g, t) + \eta_{Rr}^{Rr}(g, t)0.5 + \eta_{Rr}^{rr}(g, t)0.5 +\\
	 &~~~~\eta_{rr}^{RR}(g, t) + \eta_{rr}^{Rr}(g, t)0.5
\end{split}\\
	f_{rr}(g, t) &= \eta_{rr}^{rr}(g, t) + \eta_{rr}^{Rr}(g, t)0.5 + \eta_{Rr}^{rr}(g, t)0.5 + \eta_{Rr}^{Rr}(g, t)0.25
\end{align}  
\end{subequations}  

We close the life cycle by adding the seeds produced during time step $t$, to the seed bank (Eq. \ref{eq:above_ground}) so that 
\begin{equation}
	b_G(g, t + 1) = b_G(g, t)(1 - \phi_e)\phi_b + f_G(g, t).  
\end{equation}
Where $(1 - \phi_e)$ is the proportion of seeds that did not germinate $\phi_b$ is the probability that a seed in the seed bank, $b_G(g, t)$, survives one year. 

To test the effect dispersal limitation of both seeds and pollen had on the evolutionary dynamics we built a 1D spatial version of the model. The full spatial model is set out in \nameref{S1_Appendix}. The main difference is that we introduce a location state variable $x$ so that the seed bank becomes location specific, $b_G(g, x, t)$, and all the density state functions that arise from it ($n_G(g, x, t)$, ect.) also be location specific. Populations and different locations interact via the pollen and seed dispersal functions. The probability density function for pollen arriving at location $x$ (Eq. \ref{eq:pollen_func}) now requires integration over the pollen arriving at location $x$ from all other locations ($x_p$)  
\begin{equation}\label{eq:pollen_func_space}
\gamma_G(g, x, t) = \frac{\int_{x_p} n''_G(g, x, t)d_p(x, x_p)\text{d}x_p} {\sum_{\forall G_p}\int_{x_p}\int_{g_p} n''_{G_p}(g_p, t)d_p(x, x_p) \text{d}g_p\text{d}x_p} 
\end{equation}     
Where the pollen dispersal kernel is 
\begin{equation}\label{eq:pollen_disp}
	d_p(i, j) = \frac{c}{a^{2/c}\Gamma\left(\dfrac{2}{c} \right)\Gamma\left(1 - \dfrac{2}{c} \right)}\left( 1 + \dfrac{\delta_{i,j}^c}{a} \right)^{-1} 
\end{equation} 
This is a logistic kernel, which was found to be the one of the best fitting pollen dispersal kernels for oil seed rape \cite{Klei2006}. This is a two parameter kernel with a scale, $a$, and shape, $c$, parameter, where $\delta_{i,j}$ is the distance between locations $i$ and $j$ and $\Gamma(\cdot)$ is the gamma function. 

The probability that a seed produced at location $x$ is dispersed to location $x'$ is 
\begin{subequations}\label{eq:seed_disp}
\begin{equation}\label{eq:seed_kern}
	d_m(x, x') = \alpha \Upsilon_1 \Omega_1 \delta_{ij}^{\Omega_1 - 2} e^{-\Upsilon_1 \delta_{x,x'}^{\Omega_1}} + (1 - \alpha) \Upsilon_2 \Omega_2 \delta_{x,x'}^{\Omega_2 - 2} e^{-\Upsilon_2 \delta_{ij}^{\Omega_2}}  
\end{equation}
\begin{equation}\label{eq:shape}
	\Omega_k = \frac{1}{1 + \text{ln}(1 - \omega_k)}
\end{equation}
\begin{equation}\label{eq:scale}
	\Upsilon_k = \frac{\Omega_k - 1}{\Omega_k \mu_k^{\Omega_k}}
\end{equation}
\end{subequations} 
This double Weibull dispersal kernel was found to be the best fit to black grass seed dispersal in a majority of cases \cite{Colb2001}. $\delta_{x,x'}$ is the distance between locations $x$ and $x'$, $\alpha$ is the proportion of seeds in the short dispersal kernel, $\mu_k$ is the distance most seeds disperse to under kernel $k \in \{1, 2\}$. The skew of kernel $k$ is controlled by $\omega_k$, the proportion of seeds that disperse up to distance $\mu_k$. 
 
The model was implemented in Julia version 0.5.0, and code for the model and plotting is available in \nameref{S1_File}.
% Place figure captions after the first paragraph in which they are cited.
%\begin{figure}[!h]
%\caption{{\bf Bold the figure title.}
%Figure caption text here, please use this space for the figure panel descriptions instead of using subfigure commands. A: Lorem ipsum dolor sit amet. B: Consectetur adipiscing elit.}
%\label{fig1}
%\end{figure}

\subsection*{Study system and parametrization}
We parametrize this model using \textit{A. myosuroides} is an annual, diploid, out-crossing grass that is one of the most serious agricultural weeds in Europe \cite{Moss2007}. Due to decades of intensive herbicide use \textit{A. myosuroides} has evolved widespread target site \cite{Moss2007} and quantitative resistance \cite{Yu2014}.   

Several of the population model parameters, particularly those relating to the quantitative genetic selection model, are unknown for our study system. We used field-derived estimates of population density to calibrate the model. While we cannot directly parametrize the model with this data, we could use it to calibrate the model and constrain sets of parameter values to those that predicted population sizes in the range of observed population sizes. We outline this procedure in \nameref{S2_Appendix}. Estimates and sources for each parameter are given in Table \ref{tab:parameters}.          

\begin{table}[!ht]
\begin{adjustwidth}{-2.25in}{0in} % Comment out/remove adjustwidth environment if table fits in text column.
\centering
\caption{
{\bf Model parameters with range used in parameter filtering (see \nameref{S2_Appendix}), etimated value, brief description and source}}
\begin{tabular}{|l+l|l|p{9.5cm}|p{2.5cm}|}
\hline
		{\bf parameter} & {\bf range} & {\bf estimate} & {\bf description} & {\bf source}\\
 \thickhline
 &\multicolumn{4}{l|}{{\it Population model}}\\ \hline
	$\phi_b$ & 0.22 -- 0.79 & 0.42 & seed survival probability & \cite{Thom1997}\\ \hline
	$\phi_e$ & 0.45 -- 0.6 & 0.52 & germination probability & \cite{Colb2006}\\ \hline	
	$f_\text{max}$ & 30 -- 300$^\blacklozenge$ & 45 & seed production (in seeds/plant) of highly susceptible individuals at low densities & \cite{Doyl1986}\\ \hline
	$f_d$ & 0 -- 0.15$^\dag$ & 0.004 & reciprocal of population (1/pop.) at which individuals interfere with each others fecundity & \cite{Doyl1986}\\ \hline 
	$f_c^0$ & 4 -- 10$^\dag$ & no est$^\ddag$  & fecundity in a naive population in logit($f_0$)$f_\text{max}$ & simulation\\ \hline
	$f_c$ & $0.1f_0$ -- $2f_0 ^\dag$ & no est$^\ddag$ & reduction in fecundity (in logits) due to a one unit increase in resistance. Only meaningful in relation to $f_0$ & simulation\\ \hline
	$V_a$ & 0.5 -- 1.5 &  & Variance of the offspring distribution (additive variance) & varied to explore effect\\ \hline
	$s_0$ & fixed & 10 & survival in a naive population is logit($s_0$) & fixed, use $\xi$ and $\rho$ to control survival function\\ \hline
	&\multicolumn{4}{l|}{{\it Management effects}}\\ \hline
	$\varsigma$ & 0.5 -- 1$^\dag$ & 0.8$^\blacklozenge$ & proportion of above ground individuals exposed to herbicide & HGCA\\ \hline   		
	$\xi$ & $2s_0$ -- $3s_0^\dag$ & no est$^\ddag$ & reduction in survival (in logits) due to herbicide (only meaningful in relation to $s_0$) & simulation\\ \hline	
	$\rho$ & $0.1\xi$ -- $2\xi$ & no est$^\ddag$ & protection against herbicide (in logits) conferred by a one unit increase in $g$, only meaningful in relation to $\xi$ & simulation\\ \hline
	int$_{Rr}$ & 0.001 -- 0.2 & & initial frequency of resistant target site genotype. All other individuals are assumed to be of genotype rr & \\ \hline
	&\multicolumn{4}{l|}{{\it Dispersal}}\\ \hline
	$\alpha$ & 0.38 -- 0.58$^\dag$ & 0.48 & proportion of seeds in short dispersal kernel & \cite{Colb2001}\\ \hline   
	$\mu_1$ & 0.46 -- 0.7$^\dag$ & 0.58 & distance (in m) at which maximum number of seeds are found in short seed dispersal kernel & \cite{Colb2001}\\ \hline
	$\mu_2$ & 1.32 -- 1.98$^\dag$ & 1.65 & distance (in m) at which maximum number of seeds are found in long seed dispersal kernel & \cite{Colb2001}\\ \hline
	$\omega_1$ & 0.35 -- 0.53$^\dag$ & 0.44 & proportion of seeds that disperse up to distance $\mu_1$ in short seed dispersal kernel & \cite{Colb2001}\\ \hline
	$\omega_2$ & 0.31 -- 0.47$^\dag$ & 0.39 & proportion of seeds that disperse up to distance $\mu_2$ in long seed dispersal kernel & \cite{Colb2001}\\ \hline
	$a$ & 25.6 --38.4$^\dag$ & 32.3 & scale parameter for pollen dispersal kernel & \cite{Klei2006}\\ \hline
	$c$ & 2.66 -- 3.98$^\dag$ & 3.32 & shape parameter for pollen dispersal kernel & \cite{Klei2006}\\ \hline
\end{tabular}
\begin{flushleft} $\blacklozenge$ sourced from grey literature, unpublished data and expert opinion\\
	$\dag$ range not available from literature, simulation used to find plausible range\\
	$\ddag$ no estimate not available from literature, simulation used to find plausible range
\end{flushleft}
\label{tab:parameters}
\end{adjustwidth}
\end{table}

\subsection*{Sensitivity analysis}
The model calibration resulted in 11,866 parameter combinations that produced realistic densities. We performed a global sensitivity analysis on this set of 11,866 parameter sets to explore how different parameters, and their interactions, affected the behaviour of the model. We followed the approach of \cite{Cout2014} and fitted Boosted Regression Trees (BRTs) to find relationships between the parameter values and the model behaviours of interest. We focused on three aspects of the models behaviour: the speed at which target site resistance establishes in the population, and how much resistance was derived from quantitative resistance. We used the frequency of alleles that are $R$ at final time step $T$ (\%$R_T$) as a metric of target site resistance spread, the survival of target site susceptible individuals under herbicide ('survival $rr$') at the highest mean $g$ achieved as a metric of how important quantitative resistance was, and mean spread rate as a metric of population spread. For a detailed explanation of the sensitivity analysis see \nameref{S2_Appendix}.

\subsection*{Simulation experiments}
Because we assume no mutation in the target site resistance locus, target site resistance can only develop in a population the target site resistant allele needs to be introduced from a population where it has already developed through mutation. It is also possible to import high levels of quantitative resistance. To explore how the composition of the source and receiving populations influence the rate of evolution of resistance in the latter we created two general scenarios: a natural spread scenario, where target site resistance starts at one edge of the landscape, and is allowed to invade into rest of the landscape (Fig. \ref{fig:exper}A); and a transplant scenario (Fig. \ref{fig:exper}B) where a small number of seeds are deposited in the center of a receiving landscape. The natural spread scenario replicates spread at a smaller scales, where the target site genotype is invading into a neighbouring field (possibly under different management), or from a field margin. The transplant scenario replicates the situation when a small amount of seeds are transported to a new, distant, area by farm vehicles. 

To asses the behaviour of the model we use \%$R$ to measure the frequency of target site resistance. The level of quantitative resistance in a population was measured as the survival or $rr$ individuals ('survival $rr$'). We are also interested in the demographic advantage experienced by target site resistant individuals. We measure this as the ratio of the average number of seeds produced by a target site resistant individual that emerges from the seed bank, to the average number of seeds produced by a target site susceptible individual that emerges from the seed bank, under herbicide at location $x$ and time $t$.
\begin{subequations}
\label{eq:TSR_adv}
\begin{equation}
	K(x, t) = \frac{\kappa_R(x, t)}{\kappa_r(x, t)}
\end{equation}
\begin{equation}
	\kappa_R(x, t) = \frac{\int_g (n(g, RR, x_m, t) + n(g, Rr, x_m, t)) s(g, RR, x, h_x = 1)\psi(g, x)\text{d}g}{\int_g n(g, RR, x_m, t) + n(g, Rr, x_m, t)\text{d}g}
\end{equation}
\begin{equation}
	\kappa_r(x, t) = \frac{\int_g n(g, rr, x_m, t) s(g, rr, x, h_x = 1) \psi(g, x)\text{d}g}{\int_g n(g, rr, x_m, t)\text{d}g} 
\end{equation}
\end{subequations}
$K(x, t)$ incorporates the effect of $g$ on both fecundity and survival. When $k = 1$ both target site resistant and susceptible individuals produce the same average number of seeds over their life span. When $K > 1$ for every seed an average target site susceptible individual produces, target site resistant individuals produce $K$ seeds on average.   

Unless otherwise stated in these simulation experiments $f_0 = 4$, $s_0 = 10$, $\rho = 1.5$, $f_r = 0.45$ and $\zeta = 16$. With these values an $rr$ individual with quantitative resistance $g = 5$ will have survival of 0.85 under herbicide, but produce $f_{max} 0.85$ seeds and pollen (assuming density is low).

\begin{figure}[!h] 
	\includegraphics[width=100mm]{/home/shauncoutts/Dropbox/projects/MHR_blackgrass/BG_population_model/writting/figures/experiments_schematic_simp.pdf}
\caption{{\bf Simulation experiments.} One target site resistant seed and nine target site susceptible seeds were added to the seed bank of the receiving landscape after 20 time steps. The added seeds had either high or low levels of quantitative resistance. The receiving population was in one of three states (empty, exposed to herbicide for 20 time steps, naive to herbicide). After the 20 time step establishment period the entire population was exposed to herbicide in all three scenarios.} 
\label{fig:exper}
\end{figure}

% Results and Discussion can be combined.
\section*{Results}
The ecological and genetic background against which the invasion of the resistance genotype took place had an enormous effect on how target site and quantitative resistance interacted. When target site resistant alleles were introduced to a population that had already developed high levels quantitative resistance, target site resistance evolved very slowly (Fig. \ref{fig:nospace_time}c,d), with $R$ making up less than 5\% of all alleles after 100 time steps. When target site resistant alleles were introduced into an empty landscape (Fig. \ref{fig:nospace_time}a) or a population that had low quantitative resistance (Fig. \ref{fig:nospace_time}a) target site resistance approached fixation, although the speed at which this happened depended on the quantitative resistance of the introduced seeds and the additive genetic variance of quantitative resistance ($V_a$). As expected, when target site resistance was introduced into an empty landscape along side seeds with low quantitative resistance, it quickly approached fixation (Fig \ref{fig:nospace_time}a). And high levels of quantitative resistance never evolved. when target site resistance was introduced into an empty landscape along side seeds with high quantitative resistance, it took more than 50 time steps to approach fixation (Fig \ref{fig:nospace_time}b). And by 50 time steps high levels of quantitative and target site resistance coexisted. When target site resistant alleles where introduced into a population that had never been exposed to herbicide it approached fixation much more quickly when additive genetic variance ($V_a$) was low (lighter green Fig. \ref{fig:nospace_time}e,f), and high levels of quantitative resistance never developed (dashed lines). However, when $V_a$ was high (dark green) target site resistance evolved much more slowly and both target site and quantitative resistance co-existed over the medium term. It is also interesting to note that the type of population the target site resistance was introduced into changed the direction of the effect of $V_a$. In the empty landscape higher levels of $V_a$ speed up the evolution of target site resistance and consequently reduced the level of quantitative resistance (darker green solid lines are above lighter ones, vice versa for dashed lines Fig. \ref{fig:nospace_time}b). The opposite pattern was seen in the naive population (Fig. \ref{fig:nospace_time}f).  

\begin{figure}[!h] 
	\includegraphics[height=90mm]{/home/shauncoutts/Dropbox/projects/MHR_blackgrass/BG_population_model/TSR_rho15_injRR01.pdf}
\caption{{\bf Evolution of target site ($\%R$) and quantitative (survival $rr$) resistance under different levels of additive variance ($V_a$).} Seeds with low (a, c, e) and high (b, d, f) levels of quantitative resistance are introduced into an empty landscape (a,b), or an area that already has \textit{A. myosuroides} present, but no target site resistance alleles and high levels of quantitative resistance (c,d), or low levels of quantitative resistance (e,f).} 
\label{fig:nospace_time}
\end{figure}

This general pattern holds under a wide range of values for the resistance conferred by a one unit increase in $g$ ($\rho$); with higher additive variance ($V_a$) allowing faster evolution of target site resistance when the population already had higher levels of quantitative resistance (Fig. \ref{fig:gpro}b,c,d), and slowing the evolution of target site resistance when the population was naive to herbicide (Figure \ref{fig:gpro}e,f). 

The landscape that target site resistance was introduced to changed the values of $\rho$ where high levels of target site and quantitative resistance could co-exist. When target site resistance was introduced into empty landscape, along with a few target site susceptible seeds (Fig. \ref{fig:gpro}a,b), or a large population that had never been exposed to herbicide (Fig. \ref{fig:gpro}e,f), high levels of both target site and quantitative resistance only co-exist when $\rho$ was high. In contrast, when target site resistance was introduced into a large population that had high levels of quantitative resistance, target site and quantitative resistance co-existed at low levels of $\rho$ (Fig. \ref{fig:gpro}c,d). It is interesting to note the effect of the quantitative resistance of introduced seeds. When the 10 seeds (one being target site resistant) were introduced into an empty landscape the effect of the quantitative resistance of the introduced seeds makes a big difference(Fig. \ref{fig:gpro}a vs. b). This is expected since those seeds represent the entire initial population. More surprising, when 10 seeds were introduced into a large population that had high levels of quantitative resistance the imprint of those ten seeds were still evident after 100 time steps, especially when $V_a$ was low. When $\rho = 1$, $V_a = 0.5$ and the survival of target site susceptible introduced seeds was 0.01, $>75\%$ of target site alleles in the population were resistant after 100 time steps (light green, solid line Fig. \ref{fig:gpro}c). Under the same condition when the survival of target site susceptible introduced seeds was 0.97 (high quantitative resistance) $\%R < 60\%$ after 100 time steps. While this difference is modest, it is the effect of 10 seeds introduced into a population of 10's of thousands, still evident after 100 time steps.

\begin{figure}[!h] 
	\includegraphics[height=90mm]{/home/shauncoutts/Dropbox/projects/MHR_blackgrass/BG_population_model/TSR_quant_rho_injRR01.pdf}
\caption{{\bf Effect of $\rho$ (protective effect of $g$) on target site and quantitative resistance.} All populations had a establishment period of 20 time steps and were then exposed to herbicide for 100 time steps, after which time target site ($\%R$, solid line) and quantitative resistance (survival $rr$, dashed line) were measured. We tested different quantitative resistance for introduced seeds (columns) and receiving landscapes (rows).}        
\label{fig:gpro}
\end{figure}

When we model the evolution of target site and quantitative resistance as an explicitly spatial process target site can establish in a population that has already developed high levels of herbicide resistance (compare Fig. \ref{fig:pro_R_spr}c and Fig. \ref{fig:nospace_time} c).This could happen in the spatial model because dispersal limitation meant the target site resistant parents were more likely to cross with each other, and their offspring were dispersed to nearby locations (and thus more likely to cross with each other). Thus, dispersal limitation meant the target site resistant population could purge the unneeded fitness costs of quantitative resistance more efficiently. Further, once those fitness costs were purged, target site resistant parents crossed with target site susceptible individuals, pushing a lower breeding value for quantitative resistance into the local target site susceptible population. This purging process was more efficient when $V_a$ was higher (Fig. \ref{fig:pro_R_spr}a vs. \ref{fig:pro_R_spr}c).

\begin{figure}[!h] 
	\includegraphics[height=90mm]{/home/shauncoutts/Dropbox/projects/MHR_blackgrass/BG_population_model/TSR_space_injrr01_simp.pdf}
\caption{{\bf Spread of target site resistant allele $R$ under the spatial model, after 10 seeds (1 $RR$, 9 $rr$) where introduced at a single location.} $\%R$ at each location over time, later times are in darker blues. We show the two scenarios where target site resistance was introduced into an existing \textit{A. myosuroides} population (exposed and naive), under two different levels of additive variance ($V_a$). Introduced seeds had low quantitative resistance (survival $rr$ int = 0.01). Results from the equivalent non-spatial model are shown in Fig \ref{fig:nospace_time}c and d} 
\label{fig:pro_R_spr}
\end{figure}

As in the non-spatial model, the effect of the additive variance in quantitative resistance was reversed when target site resistant alleles were introduced into a population that was not previously exposed to herbicide, evolving and spreading faster when $V_a$ was lower (Fig. \ref{fig:pro_R_spr}b vs. \ref{fig:pro_R_spr}d).             



TO HERE

\section*{Discussion}
We find that the ecological and historical context surrounding the invasion of target site resistance into a new area can have an enormous effect on how that invasion unfolds and which genetic architecture dominates. The invasion of target site resistance alleles was much slower when the invasion was into an area that already contained \textit{A. myosuroides}. In addition the frequency of target site resistance and amount of quantitative resistance changed very quickly over space, and on the target site resistance invasion front both architectures co-existed for long periods of time. This occurred even though target site resistance conferred perfect resistance with no demographic costs, giving target site resistant individuals a competitive advantage over those individuals with only quantitative resistance. When the invasion occurred into an empty landscape its dynamics were determined the genetic architecture of the source population and the of quantitative resistance. Given this variability we might expect to see different architectures predominate in different regions of a landscape. To date there is little evidence to either support or refute this hypothesis because the prevalence and type of resistance has not typically been studied at landscape scales \cite{Dely2013}.

We find two scenarios where co-existence of both target site and quantitative resistance occurred; firstly on the invasion front when target site resistant alleles are invading into and existing population where quantitative resistance has developed. This scenario suggests that if herbicides are applied in a spatial mosaic (a strategy recommended to slow the evolution of resistance \cite{Rex2013}), there may be boarder areas where both architectures co-exist. We also found co-existence of both target site and quantitative resistance across the whole population when the invasion occurred into an empty landscape and the source population had high levels of quantitative resistance (when the protective effect of quantitative resistance was sufficiently high). This suggests co-existence will be most common at the leading edge of an invasion, where expanding nascent population foci are common \cite{Mooy1988}. Both these scenarios make it almost inevitable that both genetic architectures co-exist somewhere in the landscape, although the area may be small.

This has important implications for the management of resistance. The continuous nature of quantitative resistance means that there is always some variation for selection to act on and so it can develop earlier \cite{Dely2010newPhy}. Quantitative resistance can also give cross resistance between compounds \cite{Bauc2016, Neve2007}. While target site resistance can confer high levels of resistance for little demographic cost \cite{Bauc2016}. Places where both architectures co-exist may experience the worst of both worlds, resistance that develops very quickly, with very high levels of resistance to existing compounds \cite{Vera2015}, and cross resistance to novel compounds when they are introduced \cite{Neve2007}. These differences in behaviour also mean different strategies are used to manage target site and quantitative resistance (fore example compound cycling to delay the evolution of target site resistance \cite{Rex2013}) \cite{Gard1998}. When populations have both target site and quantitative resistance we may find ourselves with few good strategies to slow the evolution of resistance \cite{Gard1998}. 

Because we assume target site resistance is both more effective and less costly than quantitative resistance, under constant selection all resistance in the population will eventually be target site resistance. However, quantitative resistance can occur for long periods of time when it has a low demographic cost, and a moderate to strong protective effect. In our model these quantities were fixed parameters. In reality both the protective effect of quantitative resistance, and the demographic costs it incurs will also be under selection. Over time the metabolic pathways and other mechanisms of quantitative resistance that provide the most protection for the least demographic cost, will predominate. This gives reason to believe that quantitative resistance can be highly effective and have a low demographic costs [REF].    

\section*{Conclusion}
Resistance affects food production systems and human health across the global. Our results suggest that quantitative resistance can greatly slow the invasion of target site resistance, and the evolutionary dynamics within nascent population foci on the invasion front can result in a different genetic architectures dominating or co-existing. As a result we should expect the genetic architecture underlying resistance to be heterogeneous at both local and landscape scales. It follows from this that the effectiveness of different resistance management strategies, like compound cycling and very high doses, may also vary over space, as these different strategies exploit weaknesses in different genetic architectures \cite{Gard1998, Rex2013}. Our model is a first step in generating hypothesis on how the different genetic architectures that underlie resistance will interact across space. It is imperative that this work is followed by empirical observations and experimental tests. A logical starting point would be spatially extensive surveys of resistant populations to see where and when target site and quantitative resistance dominate or co-exist.

\section*{Supporting Information}

% Include only the SI item label in the paragraph heading. Use the \nameref{label} command to cite SI items in the text.
\paragraph*{S1 Fig.}
\label{S1_Fig}
{\bf Heat maps of survival of target site susceptible plants under herbicide}, a measure of the importance of quantitative resistance. These plots show the development of quantitative resistance over time and space, under three patterns of herbicide use.

\paragraph*{S2 Fig.}
\label{S2_Fig}
{\bf The frequency of target site resistant alleles (\%R) and the amount of target site and quantitative resistance coexistence, over $f_r$ and $\rho$ parameter space.} 

\paragraph*{S1 File.}
\label{S1_File}
{\bf Julia implementation of model and plotting code}  

\paragraph*{S1 Appendix.}
\label{S1_Appendix}
{\bf Spatial model.} 
Expanded explanation and worked example of the target site mixing function $Q(G, G_m, G_p)$.

\paragraph*{S1 Appendix.}
\label{S2_Appendix}
{\bf Parameter sanity check and sensitivity analysis} 

\section*{Acknowledgments}
We thank some people for some things

\nolinenumbers

% Either type in your references using
% \begin{thebibliography}{}
% \bibitem{}
% Text
% \end{thebibliography}
%
% or
% % Compile your BiBTeX database using our plos2015.bst
% style file and paste the contents of your .bbl file
% here.
% 
%\begin{thebibliography}{10}
%
%\bibitem{bib1}

%Conant GC, Wolfe KH.
%\newblock {{T}urning a hobby into a job: how duplicated genes find new
%  functions}.
%\newblock Nat Rev Genet. 2008 Dec;9(12):938--950.

%\end{thebibliography}

%REMEMBER TO FIX UP THESE CITATIONS AND USE THEIR FORMATTING ONCE I HAVE A MORE COMPLEATE VERSION.

%\bibliographystyle{/home/shauncoutts/Dropbox/shauns_paper/referencing/bes} 
\bibliography{/home/shauncoutts/Dropbox/shauns_paper/referencing/refs}


\end{document}

