% Template for PLoS
% Version 3.3 June 2016
%
% % % % % % % % % % % % % % % % % % % % % %
%
% -- IMPORTANT NOTE
%
% This template contains comments intended 
% to minimize problems and delays during our production 
% process. Please follow the template instructions
% whenever possible.
%
% % % % % % % % % % % % % % % % % % % % % % % 
%
% Once your paper is accepted for publication, 
% PLEASE REMOVE ALL TRACKED CHANGES in this file 
% and leave only the final text of your manuscript. 
% PLOS recommends the use of latexdiff to track changes during review, as this will help to maintain a clean tex file.
% Visit https://www.ctan.org/pkg/latexdiff?lang=en for info or contact us at latex@plos.org.
%
%
% There are no restrictions on package use within the LaTeX files except that 
% no packages listed in the template may be deleted.
%
% Please do not include colors or graphics in the text.
%
% The manuscript LaTeX source should be contained within a single file (do not use \input, \externaldocument, or similar commands).
%
% % % % % % % % % % % % % % % % % % % % % % %
%
% -- FIGURES AND TABLES
%
% Please include tables/figure captions directly after the paragraph where they are first cited in the text.
%
% DO NOT INCLUDE GRAPHICS IN YOUR MANUSCRIPT
% - Figures should be uploaded separately from your manuscript file. 
% - Figures generated using LaTeX should be extracted and removed from the PDF before submission. 
% - Figures containing multiple panels/subfigures must be combined into one image file before submission.
% For figure citations, please use "Fig" instead of "Figure".
% See http://journals.plos.org/plosone/s/figures for PLOS figure guidelines.
%
% Tables should be cell-based and may not contain:
% - spacing/line breaks within cells to alter layout or alignment
% - do not nest tabular environments (no tabular environments within tabular environments)
% - no graphics or colored text (cell background color/shading OK)
% See http://journals.plos.org/plosone/s/tables for table guidelines.
%
% For tables that exceed the width of the text column, use the adjustwidth environment as illustrated in the example table in text below.
%
% % % % % % % % % % % % % % % % % % % % % % % %
%
% -- EQUATIONS, MATH SYMBOLS, SUBSCRIPTS, AND SUPERSCRIPTS
%
% IMPORTANT
% Below are a few tips to help format your equations and other special characters according to our specifications. For more tips to help reduce the possibility of formatting errors during conversion, please see our LaTeX guidelines at http://journals.plos.org/plosone/s/latex
%
% For inline equations, please be sure to include all portions of an equation in the math environment.  For example, x$^2$ is incorrect; this should be formatted as $x^2$ (or $\mathrm{x}^2$ if the romanized font is desired).
%
% Do not include text that is not math in the math environment. For example, CO2 should be written as CO\textsubscript{2} instead of CO$_2$.
%
% Please add line breaks to long display equations when possible in order to fit size of the column. 
%
% For inline equations, please do not include punctuation (commas, etc) within the math environment unless this is part of the equation.
%
% When adding superscript or subscripts outside of brackets/braces, please group using {}.  For example, change "[U(D,E,\gamma)]^2" to "{[U(D,E,\gamma)]}^2". 
%
% Do not use \cal for caligraphic font.  Instead, use \mathcal{}
%
% % % % % % % % % % % % % % % % % % % % % % % % 
%
% Please contact latex@plos.org with any questions.
%
% % % % % % % % % % % % % % % % % % % % % % % %

\documentclass[10pt,letterpaper]{article}
\usepackage[top=0.85in,left=2.75in,footskip=0.75in]{geometry}

% amsmath and amssymb packages, useful for mathematical formulas and symbols
\usepackage{amsmath,amssymb}

% Use adjustwidth environment to exceed column width (see example table in text)
\usepackage{changepage}

% Use Unicode characters when possible
\usepackage[utf8x]{inputenc}

% textcomp package and marvosym package for additional characters
\usepackage{textcomp,marvosym}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}
%\usepackage{natbib}

% Use nameref to cite supporting information files (see Supporting Information section for more info)
\usepackage{nameref,hyperref}

% line numbers
\usepackage[right]{lineno}

% ligatures disabled
\usepackage{microtype}
\DisableLigatures[f]{encoding = *, family = * }

% color can be used to apply background shading to table cells only
\usepackage[table]{xcolor}
\usepackage{multirow} %connecting columns in tables
\usepackage{multicol}
\usepackage{longtable}

% array package and thick rules for tables
\usepackage{array}

% create "+" rule type for thick vertical lines
\newcolumntype{+}{!{\vrule width 2pt}}

% create \thickcline for thick horizontal lines of variable length
\newlength\savedwidth
\newcommand\thickcline[1]{%
  \noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
  \cline{#1}%
  \noalign{\vskip\arrayrulewidth}%
  \noalign{\global\arrayrulewidth\savedwidth}%
}

% \thickhline command for thick horizontal lines that span the table
\newcommand\thickhline{\noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
\hline
\noalign{\global\arrayrulewidth\savedwidth}}


% Remove comment for double spacing
%\usepackage{setspace} 
%\doublespacing

% Text layout
\raggedright
\setlength{\parindent}{0.5cm}
\textwidth 5.25in 
\textheight 8.75in

% Bold the 'Figure #' in the caption and separate it from the title/caption with a period
% Captions will be left justified
\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}
\renewcommand{\figurename}{Fig}

% Use the PLoS provided BiBTeX style
\bibliographystyle{plos2015}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother

% Leave date blank
\date{}

% Header and Footer with logo
\usepackage{lastpage,fancyhdr,graphicx}
\usepackage{epstopdf}
\pagestyle{myheadings}
\pagestyle{fancy}
\fancyhf{}
\setlength{\headheight}{27.023pt}
\lhead{\includegraphics[width=2.0in]{PLOS-submission.eps}}
\rfoot{\thepage/\pageref{LastPage}}
\renewcommand{\footrule}{\hrule height 2pt \vspace{2mm}}
\fancyheadoffset[L]{2.25in}
\fancyfootoffset[L]{2.25in}
\lfoot{\sf PLOS}

%% Include all macros below

\newcommand{\lorem}{{\bf LOREM}}
\newcommand{\ipsum}{{\bf IPSUM}}

%% END MACROS SECTION


\begin{document}
\vspace*{0.2in}

% Title must be 250 characters or less.
\begin{flushleft}
{\Large
\textbf\newline{Eco-evolutionary Dynamics on the Invasion Front Drives the Co-existence of Target Site and Quantitative Resistance} % Please use "title case" (capitalize all terms in the title except conjunctions, prepositions, and articles).
}
\newline
% Insert author names, affiliations and corresponding author email (do not include titles, positions, or degrees).
\\
Name1 Surname\textsuperscript{1,2\Yinyang},
Name2 Surname\textsuperscript{2\Yinyang},
Name3 Surname\textsuperscript{2,3\textcurrency},
Name4 Surname\textsuperscript{2},
Name5 Surname\textsuperscript{2\ddag},
Name6 Surname\textsuperscript{2\ddag},
Name7 Surname\textsuperscript{1,2,3*},
with the Lorem Ipsum Consortium\textsuperscript{\textpilcrow}
\\
\bigskip
\textbf{1} Affiliation Dept/Program/Center, Institution Name, City, State, Country
\\
\textbf{2} Affiliation Dept/Program/Center, Institution Name, City, State, Country
\\
\textbf{3} Affiliation Dept/Program/Center, Institution Name, City, State, Country
\\
\bigskip

% Insert additional author notes using the symbols described below. Insert symbol callouts after author names as necessary.
% 
% Remove or comment out the author notes below if they aren't used.
%
% Primary Equal Contribution Note
\Yinyang These authors contributed equally to this work.

% Additional Equal Contribution Note
% Also use this double-dagger symbol for special authorship notes, such as senior authorship.
\ddag These authors also contributed equally to this work.

% Current address notes
\textcurrency Current Address: Dept/Program/Center, Institution Name, City, State, Country % change symbol to "\textcurrency a" if more than one current address note
% \textcurrency b Insert second current address 
% \textcurrency c Insert third current address

% Deceased author note
\dag Deceased

% Group/Consortium Author Note
\textpilcrow Membership list can be found in the Acknowledgments section.

% Use the asterisk to denote corresponding authorship and provide email address in note below.
* shaun.coutts@gmail.com

\end{flushleft}
% Please keep the abstract below 300 words
\section*{Abstract}
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur eget porta erat. Morbi consectetur est vel gravida pretium. Suspendisse ut dui eu ante cursus gravida non sed sem. Nullam sapien tellus, commodo id velit id, eleifend volutpat quam. Phasellus mauris velit, dapibus finibus elementum vel, pulvinar non tellus. Nunc pellentesque pretium diam, quis maximus dolor faucibus id. Nunc convallis sodales ante, ut ullamcorper est egestas vitae. Nam sit amet enim ultrices, ultrices elit pulvinar, volutpat risus.


% Please keep the Author Summary between 150 and 200 words
\section*{Author Summary}

\linenumbers

% Use "Eq" instead of "Equation" for equation citations.
\section*{Introduction}
There is currently a global resistance crisis \cite{Serv2013, Ross2014} and important antibiotics, pesticides and herbicides are losing their efficacy \cite{Palu2001}. Together, these chemical tools are a crucial part of the worlds food productions system \cite{Duke2012}, and are used worldwide to control life threatening bacterial infections and diseases spread by insects \cite{Nkya2013}, however, evolving resistance threatens their continued use \cite{Barb2011, Nkya2013}. Economic and regulatory conditions mean that bringing new, safe, effective, compounds to market is time consuming and expensive \cite{Duke2012}. In addition, the useful life of new chemical tools can be as short as three years if their use is not well managed \cite{Palu2001, Duke2012}. To address this crisis and reduce its impact we need to understand where and when resistance will evolve, and under what conditions. 

Until recently, target site resistance, conferred a single gene of large effect, has formed the basis of our understanding of resistance, and the evolution of target site resistance is well understood \cite{Neve2007}. However, evolutionary biologists have recognized a continuum in the genetic architecture underlying any given trait since the early 20th century (genetic architecture \textit{sensu} \cite{Deba2015}, the number and effect size of loci contributing to a trait). At one extreme traits are controlled by a single loci of large effect (as in the case of target site resistance), and at the other polygenic, quantitative traits, are controlled by smaller additive effects of many genes \cite{Land1989, Mack2009, Rajo2013}. There is growing evidence that within the same population, resistance can be conferred by genetic architectures at both extremes \cite{Donn2009, Bing2011, Hend2013, Oake2013, Bauc2016}. The implications for the evolution of these multi-architecture traits remains largely unexplored, and how multi-architecture traits interact over space is not well understood (see \cite{Deba2015}, \cite{Yeam2015} for exceptions). Such an understanding will be crucial to predict how target site and quantitative resistance interact to drive the evolution and spread of resistant genotypes through populations.

Target site resistance typically confers very high levels of resistance, and incurs very few life history costs (i.e. reductions in survival, growth and/or fecundity). Quantitative resistance is usually achieved by metabolizing the poison, or transporting the poison away from its binding site \cite{Bauc2016} and tends to confer lower levels of resistance and incur life history costs \cite{Bauc2016}. It is currently unknown how these two genetic architectures interact in the evolution of resistance. Naively, we might expect the architecture that evolves first to reduce the selective pressure for the second architecture, and so slow its evolution. However, there is some evidence from misquote control programs that low levels of resistance conferred via an inefficient quantitative pathway can increase the rate of evolution for a much less costly target site mutation that confers high levels of resistance \cite{Vera2015}.

In many cases, herbicide resistance does not evolve in all the places where the trait is expressed; instead, resistance is often imported via pollen or seeds (ref). We expect this phenomena to affect target site resistance and quantitative resistance differently. Target site resistance can first enter a population in two ways: either it can be imported, which will often result in a small number of initial target site resistance individuals, perhaps just a single seed; or it can arise \textit{in situ} through mutation at a single loci. Both long distance dispersal and mutations are rare events, and so it is expected that initially target site resistance will be a rare and binary trait (either an individual is resistant or not). This will result in very little initial standing variation in target site resistance for selection to act upon, leading to a slow spread through the population once introduced. Conversely, Quantitative resistance is thought to co-opt existing stress tolerance strategies (ref), as a result all individuals in a population will have some level of quantitative resistance, although that level might be very low. Further, since quantitative resistance is regulated by multiple genes with additive effects, that quantitative resistance will be approximately normally distributed \cite{Land1989, Mack2009, Rajo2013}. This means that even naive populations that have never before been exposed to herbicide will have standing variation in quantitative resistance on which selection can act. These differences suggest that importation of resistance may be more important for target site resistance. Clumping of genotypes (due to dispersal limitation in seeds and pollen) could allow target site resistance to take hold more quickly, since individuals with rare genotypes (like initial target site resistant mutants) will be more likely to cross with other rare mutants if closely related individuals exist spatially close to each other (i.e. sibling-sibling crosses and parent-child crosses). However, once target site resistance is established, dispersal limitation will restrict its spread through the population. 

We develop one of the first spatially explicit, density dependent population models of the evolution of resistance as a complex, duel architecture trait. We apply this model to the economically important weed, \textit{Alopecurus myosuroides} (Huds.). Specifically, we test if quantitative resistance can slow the spread of target site resistance into a population, and if so under which range of costs for quantitative resistance. We also test how gene dispersal affects where and when both target site and quantitative resistance develop in the same population. 

\section*{Materials and Methods}
% For figure citations, please use "Fig" instead of "Figure".
\subsection*{model}
We model the invasion of a target site resistant genotype into a population of \textit{A. myosuroides} where quantitative resistance can also develop. \textit{A. myosuroides} is an annual, out-crossing grass, and we use a flexible, discreet time, continuous space modelling framework that can track the evolution of both continuous (quantitative resistance) and discreet (target site resistance) traits, called Integral Projection Models \cite{Elln2006}.     

We model a population of \textit{A. myosuroides} on a 1D landscape using a yearly time step, starting at the beginning of the growing season before any seeds have emerged from the seed bank. Once seeds emerge they are either exposed to herbicide or not, which affects their survival. Those individuals that survive then flower and spread pollen. Finally, survivors disperse their seeds into the seed bank. Survival under herbicide is conferred by two type of resistance, quantitative and target site. The quantitative resistance trait (conferred by the variable $g$). All individuals have some value for $g$, but that value may be very small, and so phenotypically individuals are susceptible. We assume target site resistance is controlled by a single loci, with inheritance following simple two allele gene and random mating. We denote the resistant allele $R$ and the susceptible allele $r$. 

All seeds in the seed bank at location $x$, time $t$, with target site genotype $G$ are distributed over metabolic resistance score $g$, we denote the distribution  
\begin{equation}\label{eq:seedbank_simp}
	b(g, G, x, t + 1) = b(g, G, x, t)(1 - \phi_e)\phi_b + f(g, G, h, x, t). 
\end{equation}
The first term in Eq. \ref{eq:seedbank_simp} is the distribution of surviving seeds in the seed bank of genotype $G$ at location $x$, where $\phi_e$ is the probability that a seed germinates and $\phi_b$ is the probability that a seed in the seed bank survives one year. The second term, $f(g, G, h, x, t)$, is the distribution of new seeds of genotype $G$ added to the population at location $x$. Note that $b(g, G, x, t)$ and $f(g, G, h, x, t)$ are distributions over metabolic resistance score $g$ and locations $x$. To get the total number of seeds for genotype $G$ these distributions have to be integrated over $g$ and $x$. Also note that we expect the mean of each target site genotypes' distribution over $g$ to be different, since target site resistant individuals can have low levels of quantitative resistance and still survive herbicide application without the demographic costs associated with quantitative resistance. 

Each genotype can contribute both target site resistance alleles and quantitative resistance genes to the other genotypes at all other locations through       
\begin{align}
\label{eq:fecund}
\begin{split}
	f(&g, G, h, x', t) = \displaystyle \sum_{\forall G_m}\sum_{\forall G_p} Q(G, G_m, G_p) \int_{x_m}\int_{x_p}\int_{g_m}\int_{g_p} \text{N}(0.5 g_m + 0.5 g_p, \sigma_f)\cdot\\
	&n(g_m, G_m, x_m, t)s(g_m, G_m, x_m, h_x) \psi(g_m, x_m) d_m(x_m, x') \gamma(g_p, G_p, x_p, x_m, h_x, t)\cdot\\
	&\text{d}g_m \text{d}g_p \text{d}x_m \text{d}x_p,
\end{split}
\end{align} 
the distribution of new seeds over metabolic resistance score $g$ and target site genotype $G$, arriving at location $x'$ at time $t$ 

TO HERE
          
and is the result of three mixing kernels. The first mixes genes within and between target site resistance genotypes. This mixing kernel comprises a double summation over maternal and paternal target site genotypes ($G_m$ and $G_p \in \{\{R, R\}, \{R, r\}, \{r, r\} \}$) and the mixing function $Q(G, G_m, G_p)$. The second mixing kernel (double integration) gives the expected distribution of seeds over metabolic resistance score $g$, given the maternal and paternal parent distributions for each combination of target site resistance genotypes. Thus Eq. \ref{eq:TSR_mixing_kern} mixes the quantitative genetic trait with and between each parental target site genotype. The target site mixing function is
\begin{subequations}
\begin{equation}\label{eq:TSR_mixing_kern}
	Q(G, G_m, G_p) = \frac{\sum_{\forall k \in G_m \times G_p} q(G, k)}{card\left( G_m \times G_p \right)}
\end{equation}      
\begin{equation}\label{eq:allel_count}
	q(G, k) = \begin{cases}
		1 &\text{if } a_1 \in k \bigwedge a_2 \in k\\
		0 &\text{otherwise} 
	\end{cases}, \text{where } G = \{a_1, a_2\}
\end{equation} 
\end{subequations}
This function uses set notation that may be unfamiliar so I will go through it carefully and give an example. $Q(G, G_m, G_p)$ is the fraction of seeds produced of target site genotype $G$ by parents of target site genotype $G_m$ and $G_p$. We assume that target site resistance is a single loci trait. Thus, each genotype can be thought of as a set of two alleles, $G = \{a_1, a_2\}$, with $a_i \in \{R, r \}$ and full the set of target site resistant genotypes is $G \in \{\{R, R\}, \{R, r\}, \{r, r\} \}$. The numerator is the number of all possible combinations of $G_m\{a_i\}$ and $G_p\{a_j\}$ that result in genotype $G$. To achieve this we use a structure called a Cartesian product. The Cartesian product of sets $X$ and $Y$ ($X \times Y$) is the set of all ordered pairs $(x, y)$, with $x \in X$, $y \in Y$ and $(x, y) = (x', y')$ if and only if $x = x'$ and $y = y'$. To give a more concrete example we explicitly write out the Cartesian product when $G_m = \{R, r\}$ and $G_p = \{R, r\}$  
\begin{equation*}
	\{R, r\} \times \{R, r\} = \{(R, R), (R, r), (r, R), (r, r)\}
\end{equation*}
The task is now to find the number of these ordered pairs which contain both alleles present in the genotype of interest, $G$. To do this we use the function $q(G, k)$. Recall that $G = \{a_1, a_2\}$ and note that the variable $k$ is iterated over $G_m \times G_p$, which is the set of all possible ordered pairs of alleles in $G_m$ and $G_p$. To extend the example above if the genotype of interest is $\{R, r\}$ then 
\begin{equation*}
	\sum_{\forall k \in \{R, r\} \times \{R, r\}} q(\{R, R \}, k) = 2
\end{equation*}    
Since only two of the ordered pairs, $(R, r)$ and $(r, R)$ contain both $a_1 = R$ and $a_2 = r$. To make this a proportion of all possible combinations we divide by the cardinality of the Cartesian product, $card \left( G_m \times G_p \right)$. For example 
\begin{equation*}
	card\left( \{R, r\} \times \{R, r\} \right) = card\left( \{(R, R), (R, r), (r, R), (r, r)\} \right) = 4
\end{equation*}

Now we have the proportion of genotype $G$ seeds produced by each combination of parental genotypes $G_m$ and $G_p$, but we still need the distribution of quantitative trait $g$ for the offspring of each combination of parental genotypes $G_m$ and $G_p$, at each location $x$. We do the quantitative trait mixing with a double integration in Eq. \ref{eq:fecund} over every combination of metabolic resistance score $g$ from the maternal parent distribution, $g_m$, and paternal distribution, $g_p$. The offspring produced by every pair of $g_m$:$g_p$ values are assumed to be normally distributed over $g$ with a mean of $0.5g_m + 0.5g_p$ and a standard deviation of $\sigma_f$ ($\text{N}(\cdot)$ in Eq. \ref{eq:fecund}). The parent distribution for each target site genotype $G$ at location $x$ is $n(g, G, x, t)s(g, G, x, t)$, where 
\begin{equation}\label{eq:above_ground}
	n(g, G, x, t) = b(g, G, x, t)\phi_e\phi_b
\end{equation}
is the number of individuals of target site genotype $G$ that emerge from the seed bank and establish. Note that because $b(g, G, x, t)$ is a distribution so is $n(g, G, x, t)$. The distribution of these emerged individuals that survive, $s(g, G, x, h_x)$, is based in their target site genotype, $G$, their metabolic resistance score, $g$, their location $x$, and whether or not herbicide is applied at location $x$, $h_x \in \{0, 1\}$.   
\begin{equation}\label{eq:sur_G}
	s(g, G, x, h_x) = \begin{cases} 
		\frac{1}{1 + e^{-s_0^x}} &\text{~if~} G \in G^* \\
		\frac{(1 - \varsigma)}{1 + e^{-s_0^x}} + \frac{\varsigma}{1 + e^{-\left(s_0^x - h_x\left(\xi - \textbf{min}(\xi, \rho g) \right)\right)}} &\text{~otherwise~} 		
	\end{cases} 
\end{equation}
We assume that seeds which germinate but die before seed set of non-management causes are subsumed into the seed survival term, $\phi_b$. We also assume that any reduction in survival due to increased density is subsumed into the affect of density on fecundity (Eq. \ref{eq:seed_production}). There are two ways individuals can avoid being affected by herbicide; i) they can not be exposed due to emerging later in the growing season, or they may simply be missed spatially, $\varsigma$ is the proportion of individuals exposed to the herbicide, ii) target site resistant individuals are assumed to be completely protected from the effects of the herbicide. $G^*$ is the sub-set of genotypes that give target site resistance. If $G^* \in \{\{R, R\}, \{R, r\}\}$ then target site resistance is a dominant trait and if $G^* = \{R, R\}$ then target site resistance is recessive. $s_0^x$ is the survival probability (in logits) when there is no herbicide, or for individuals not affected by herbicide, at location $x$ for individuals with resistance score $g = 0$. Notice that when $h_x = 0$ the top and bottom condition of Eq. \ref{eq:sur_G} are the same. $\xi$ is the reduction in survival (in logits) caused by herbicide for individuals with a resistance score $g = 0$ and $\rho$ is the protective effect of a one unit increase in resistance score $g$. Because $\rho$ is only meaningful in relation to $\xi$, and $\xi$ is only meaningful relative to $s_0^x$, we can fix $s_0^x \in \{10, -10\}$, for suitable and unsuitable habitat respectively, and still produce the full range of possible behaviours by choosing different values for $\rho$ and $\xi$. 

The survival due to metabolic resistance (second condition in Eq. \ref{eq:sur_G}) interacts with survival due to target site resistance. Individuals with no target site resistance will have some level of metabolic herbicide resistance (although that level may be very low). When metabolic resistance is low the difference in survival between target site resistant and non-resistant individuals (and thus the selection pressure), will be large. However, for individuals with high levels of metabolic resistance the difference in survival between target site resistant and non-resistant individuals will be small. Also metabolic resistance comes with a demographic cost, which we model as a reduction in fecundity. Target site resistant individuals require no metabolic resistance to survive herbicide application, and thus can have higher fecundity, potentially helping to drive target site resistance through the population. The number of seeds produced per individual, $\psi(g)$, is a function of resistance and the density of surviving plants, with greater resistance and higher density reducing the number of seeds. 
\begin{subequations}
\begin{equation}\label{eq:seed_production}
	\psi(g, x) = \frac{\frac{1}{3}f_\text{max}}{1 + \Psi(g) + f_d M(h_x, x, t) + f_dM(h_x, x, t) \Psi(g)}
\end{equation}  
\begin{equation}
	\Psi(g) = 1 + e^{-(f_0 - f_r|g|)}
\end{equation}
\end{subequations}
where $f_\text{max}$ is the maximum possible number of seeds per individual, $f_0$ controls the number of seeds produced when $g = 0$, $|g|$ is the absolute value of $g$, $f_r$ is the cost of resistance in terms of reduction in seed production, $1/f_d$ is the population level where individuals start to interfere with each other and 
\begin{equation}\label{eq:num_sur}
   M(h_x, x, t) = \sum_{\forall G} \int_g n(g, G, x, t)s(g, G, x, h_x)\text{d}g
\end{equation}
is the number of above ground individuals that survive until seed set at location $x$. We assume that pollen production is not affected by either density or metabolic resistance score $g$. Because pollen is more dispersive than seeds this assumption means that high quantitative resistance genotypes can spread more easily across the landscape.   

The third mixing kernel mixes both target site and metabolic resistance genes spatially. Spatial mixing also requires double integration over the location of the mother, $x_m$, and father, $x_p$. This double integration over spatial locations takes every pair of locations $x_m:x_p$ and generates a distribution of seeds over metabolic resistance score $g$ for every genotype $G$ at maternal location $x_m$ based on the frequency of pollen arriving at site $x$ with the genotype of interest ($g_p$ and $G_p$). This frequency is calculated as
\begin{equation}\label{eq:pollen_func}
\gamma(g_p, G_p, x_p, x_m, h_x, t) = \frac{n(g_p, G_p, x_p, t)
s(g_p, G_p, x_p, h_x) d_p(x_p, x_m)} {\sum_{\forall G}\int_{x}\int_{g} n(g, G, x, t) s(g, G, x, h_x) d_p(x, x_m) \text{d}g\text{d}x} 
\end{equation}

 the paternal parent distribution of survivors at paternal location $x_p$. The paternal parental distribution of survivors is weighted by the distance between the maternal and paternal locations $x_m$ and $x_p$ according to the pollen dispersal kernel 
\begin{equation}\label{eq:pollen_disp}
	d_p(i, j) = \frac{c}{a^{2/c}\Gamma\left(\dfrac{2}{c} \right)\Gamma\left(1 - \dfrac{2}{c} \right)}\left( 1 + \dfrac{\delta_{i,j}^c}{a} \right)^{-1} 
\end{equation} 
This is a logistic kernel, which was found to be the one of the best fitting pollen dispersal kernels for oil seed rape \cite{Klei2006}. This is a two parameter kernel with a scale, $a$, and shape, $c$, parameter, where $\delta_{i,j}$ is the distance between locations $i$ and $j$. For each maternal location $x_m$ the amount of each pollen type (i.e. a unique value of $g$ and $G$) arriving is divided by the total amount of pollen from every target site resistance genotype, from every location for every metabolic resistance score $g$ that arrives at the maternal location $x_m$ (double integration in denominator of Eq. \ref{eq:fecund}). 

The probability that a seed produced at maternal location $x_m$ is dispersed to location $x'$ is 
\begin{subequations}\label{eq:seed_disp}
\begin{equation}\label{eq:seed_kern}
	d_m(i, j) = \alpha \Upsilon_1 \Omega_1 \delta_{ij}^{\Omega_1 - 2} e^{-\Upsilon_1 \delta_{ij}^{\Omega_1}} + (1 - \alpha) \Upsilon_2 \Omega_2 \delta_{ij}^{\Omega_2 - 2} e^{-\Upsilon_2 \delta_{ij}^{\Omega_2}}  
\end{equation}
\begin{equation}\label{eq:shape}
	\Omega_k = \frac{1}{1 + \text{ln}(1 - \omega_k)}
\end{equation}
\begin{equation}\label{eq:scale}
	\Upsilon_k = \frac{\Omega_k - 1}{\Omega_k \mu_k^{\Omega_k}}
\end{equation}
\end{subequations} 
This double Weibull dispersal kernel was found to be the best fit to black grass seed dispersal in a majority of cases \cite{Colb2001}. $\delta_{ij}$ is the distance between locations $i$ and $j$, $\alpha$ is the proportion of seeds in the short dispersal kernel rather than the long dispersal kernel, $\mu_k$ is the distance most seeds disperse to under kernel $k \in \{1, 2\}$. The skew of kernel $k$ is controlled by $\omega_k$, the proportion of seeds that disperse up to distance $\mu_k$. 

\subsection*{Parametrization}
Several of the population model parameters, particularly those relating to the quantitative genetic selection model, are unknown for our study system. However, we do have field observations with estimates of above ground plant densities and susceptibility to herbicide. While we cannot directly parametrize the model with this data, we can use it to constrain the parameter space to regions that produce sensible results (i.e. those that match up with the observed densities). We use Latin hypercube sampling to evenly sample 24,000 parameter combinations across the parameter space. We run each set of parameters for 50 time steps with constant herbicide use, starting with an initial population of three seeds, one in each of the center three locations, with an initial frequency if the target site resistant genotype Rr, set by int$_{Rr}$. For each run we record the above ground post herbicide population after 50 time steps. From this dataset of 24,000 parameter combinations we filter out those parameter combinations that resulted in unrealistic population dynamics. We use data from 140 fields across England to define what realistic, resistant, populations look like. These fields were visually inspected in ??? - ???, and every 20 m by 20 m grid square was assessed as being in one of four density classes; 'absent' (0 plants/20m$^2$), 'low' (1 -- 160 plants/20m$^2$), 'medium' (160 -- 450 plants/20m$^2$), 'high'(450 -- 1450 plants/20m$^2$) and very high ($>$1450 plants/20m$^2$), coded as \{0, 1, 2, 3, 4\}. The resistance status of each field to the three most common herbicide active ingredients was determined in a glass house trial. Seeds from 100 seed heads from 10 different locations in each field were taken post herbicide application (July -- August). These seeds were raised in a glass house and divided into three groups of 12 -- 19 plants (median 18) for each field. Each group had ??? of the herbicide 'Atlantis', 'Fenoxaprop' or 'Cycloxydim' applied, mortality and any damage were recorded after ??? days. We assume that after 50 years of continuous herbicide use black grass population will be resistant, thus, we only included highly resistant fields, where survival to all herbicides was $>80\%$ (???? fields). To set the lower and upper number of blackgrass plants in winter wheat per hectare, after herbicide application, we use the mean density class of the highly resistant fields with the minimum (0.94) and maximum (4) observed mean density class. We then used the observed number of plants per 20m$^2$for each density class (calculated from \cite{Quee2011}) and a randomization process (see appendix 1) to determine that post herbicide populations should fall between 16,348 and 132,000 plants per ha. Because our model is run on a 1D landscape, while the data are given in plants/m$^2$, estimated at a 20m resolution, we assume the modelled population occurs in a 20 m x 500 m, 1 ha field. Note that this process is not intended to definitively parametrize the model, rather is acts as a sanity check on the model outcome, removing parameter combinations that result in black grass which are never observed in the field data. Only four parameters determined if a parameter set passed this sanity check, $f_d$, $f_\text{max}$, $f_r$ and $\phi_b$ see appendix 1 for more details.

For this approach to work well we need to constrain the parameter space as much as possible \textit{a prior}. Estimates and sources for each parameter are given in Table \ref{tab:parameters}.          

\begin{longtable}[h]{p{1.5cm} p{2cm} p{2.1cm} p{1.5cm} p{3.5cm} p{3cm}} \label{tab:parameters}\\
\caption{System model parameters}\\
	\hline
	\textbf{parameter} & \textbf{units} & \textbf{range} & \textbf{estimate} & \textbf{description} & \textbf{source}\\
	\hline
	\multicolumn{6}{l}{\underline{Population model}}\\
	$\phi_b$ & prob. & 0.22 -- 0.79 & 0.45 & seed survival & \cite{Thom1997}\\
	$\phi_e$ & prob. & 0.45 -- 0.6 & 0.52 & germination probability & \cite{Colb2006}\\	
	$f_\text{max}$ & seeds/plant & 30 -- 300$^\blacklozenge$ & 45 & seed production of highly susceptible individuals at low densities & \cite{Doyl1986}\\
	$f_d$ & 1 / pop. & 0 -- 0.15$^\dag$ & 0.004 & reciprocal of population at which individuals interfere with each other & \cite{Doyl1986}\\ 
	$f_0$ & logits & 5 -- 10$^\dag$ & no est$^\ddag$  & fecundity in a naive population is logit($f_0$)$f_\text{max}$ & simulation\\
	$f_r$ & logits & $0.1f_0$ -- $2f_0 ^\dag$ & no est$^\ddag$ & reduction in fecundity due to a one unit increase in resistance. Only meaningful in relation to $f_0$ & simulation\\
	$\sigma_f$ & arbitrary & fixed & 1 & standard deviation of offspring distribution & fixed without loss of generality\\
	$s_0$ & logits & fixed & 10 & survival in a naive population is logit($s_0$) & fixed without loss of generality\\
	\multicolumn{6}{l}{\underline{Management effects}}\\
	$\varsigma$ & prop. & 0.5 -- 1$^\dag$ & 0.8$^\blacklozenge$ & proportion of above ground individuals exposed to herbicide & HGCA\\   		
	$\xi$ & logits & $2s_0$ -- $3s_0^\dag$ & no est$^\ddag$ & reduction in survival due to herbicide (only meaningful in relation to $s_0$ & simulation\\	
	$\rho$ & logits & $0.1\xi$ -- $2\xi$ & no est$^\ddag$ & protection against herbicide conferred by a one unit increase in $g$, only meaningful in relation to $\xi$ & simulation\\
	int$_{Rr}$ & proportion & 0.001 -- 0.2 & & initial frequency of resistant target site genotype. All other individuals are assumed to be of genotype rr & simulation.'\\
	\multicolumn{6}{l}{\underline{Dispersal}}\\
	$\alpha$ & prop. & 0.38 -- 0.58$^\dag$ & 0.48 & proportion of seeds in short dispersal kernel & \cite{Colb2001}\\   
	$\mu_1$ & m & 0.46 -- 0.7$^\dag$ & 0.58 & distance at which maximum number of seeds are found in short seed dispersal kernel & \cite{Colb2001}\\
	$\mu_2$ & m & 1.32 -- 1.98$^\dag$ & 1.65 & distance at which maximum number of seeds are found in long seed dispersal kernel & \cite{Colb2001}\\
	$\omega_1$ & prop. & 0.35 -- 0.53$^\dag$ & 0.44 & proportion of seeds that disperse up to distance $\mu_1$ in short seed dispersal kernel & \cite{Colb2001}\\
	$\omega_2$ & prop. & 0.31 -- 0.47$^\dag$ & 0.39 & proportion of seeds that disperse up to distance $\mu_2$ in long seed dispersal kernel & \cite{Colb2001}\\
	$a$ & & 25.6 --38.4$^\dag$ & 32.3 & scale parameter for pollen dispersal kernel & \cite{Klei2006}\\
	$c$ & & 2.66 -- 3.98$^\dag$ & 3.32 & shape parameter for pollen dispersal kernel & \cite{Klei2006}\\
	\hline
	\multicolumn{6}{l}{$\blacklozenge$ sourced from grey literature, unpublished data and expert opinion}\\
	\multicolumn{6}{l}{$\dag$ range not available from literature, simulation used to find plausible range}\\
	\multicolumn{6}{l}{$\ddag$ no estimate not available from literature, simulation used to find plausible
	 range}
\end{longtable}

\subsection*{Sensitivity analysis}
The sanity check resulted in 11,866 parameter combinations that produced black grass populations which fell in the acceptable range. We performed a global sensitivity analysis on this set of 11,866 'realistic' parameter sets to explore how different parameters, and their interactions, affected the behaviour of the model. We followed the approach of \cite{Cout2014} and fit Boosted Regression Trees (BRTs) to the dataset of 11,710 sanity checked parameter combinations to find relationships between the parameter values and the model behaviours of interest. We are primarily interested in three aspects of the models behaviour. We measure the speed at which target site resistance establishes in the population as the time step, $t$, at which the resistant target site allele, R, makes up 50\% of all target site alleles in the population  
\begin{equation}\label{eq:t_R50}
	t_{R50} = \text{first } t \text{ when } \frac{\left(2\int_x\int_g b(g, \text{RR}, x, t) + \int_x\int_g b(g, \text{Rr}, x, t) \right)\text{d}x\text{d}g}{2\sum_{\forall G} \int_x\int_g b(g, G, x, t)\text{d}x\text{d}g} \geq 0.5
\end{equation}
Secondly we measure how quickly the black grass population spread across the landscape as $\overline{C}$, the mean number of new locations with $\geq$ one seed in the seed bank at each time step. So that slow expansion due to a full landscape does not affect the estimate of spread speed the mean is only taken over time steps where $<$90\% of locations have $\geq 1$ seed in the seed bank. Finally we measure the relative contributions of metabolic resistance and target site resistance to the survival of black grass under herbicide as   
\begin{equation}\label{eq:mean_pro_rr}
	\overline{P}_{rr} = \frac{\sum_{t=1}^{t=T}\int_x\int_g n(g, \text{rr}, x, t)s(g, \text{rr}, x, h_x = 1) - n(g, \text{rr}, x, t)\dfrac{1 - \varsigma}{1 + e^{-s_0}}\text{d}g\text{d}x}{\sum_{t=1}^{t=T}\sum_{\forall G}\int_x\int_g n(g, G, x, t)s(g, G, x, h_x = 1) - n(g, G, x, t)\dfrac{1 - \varsigma}{1 + e^{-s_0}}\text{d}g\text{d}x}
\end{equation} 
$\overline{P}_{rr}$ is the mean proportion of individuals of target site genotype rr that survived until seed production under herbicide use, corrected for the number of individuals not exposed to herbicide ($1 - \varsigma$), taken over 50 time steps. If $\overline{P}_{rr} = 0$ then all of the individuals exposed to herbicide that survived until seed set did so due to target site resistance and if $\overline{P}_{rr} = 1$ then all survival in exposed individuals until seed production was attributable to metabolic resistance.   

\subsection*{Simulation experiments}
Our aim is to generate hypotheses about how the initial conditions of the invasion of a new resistance genotype (defined by both $G$ and $g$) can change how that invasion develops over time. It is assumed that all plants have some level of quantitative resistance, since many of the known mechanisms behind quantitative resistance are generalized mechanisms for coping with stress and chemicals produced by fungi, bacteria and other plants. Thus, in some senses there is no such thing as a quantitatively susceptible individual, just individuals with low levels of quantitative resistance. In contrast target site resistance can be absent from a population. Our model assumes no mutations in the target site resistance genotype. For target site resistance to develop in a population the target site resistant allele needs to be introduced from a population where it has already developed through mutation. It is also possible to import high levels of quantitative resistance via seed. 

To explore how characteristics of the source population and receiving area affect the development of both target site and quantitative resistance over time we set up three scenarios for source populations and four scenarios for the receiving area.        

% Place figure captions after the first paragraph in which they are cited.
\begin{figure}[!h]
\caption{{\bf Bold the figure title.}
Figure caption text here, please use this space for the figure panel descriptions instead of using subfigure commands. A: Lorem ipsum dolor sit amet. B: Consectetur adipiscing elit.}
\label{fig1}
\end{figure}

% Results and Discussion can be combined.
\section*{Results}
Nulla mi mi, venenatis sed ipsum varius, Table~\ref{table1} volutpat euismod diam. Proin rutrum vel massa non gravida. Quisque tempor sem et dignissim rutrum. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi at justo vitae nulla elementum commodo eu id massa. In vitae diam ac augue semper tincidunt eu ut eros. Fusce fringilla erat porttitor lectus cursus, vel sagittis arcu lobortis. Aliquam in enim semper, aliquam massa id, cursus neque. Praesent faucibus semper libero.

% Place tables after the first paragraph in which they are cited.
\begin{table}[!ht]
\begin{adjustwidth}{-2.25in}{0in} % Comment out/remove adjustwidth environment if table fits in text column.
\centering
\caption{
{\bf Table caption Nulla mi mi, venenatis sed ipsum varius, volutpat euismod diam.}}
\begin{tabular}{|l+l|l|l|l|l|l|l|}
\hline
\multicolumn{4}{|l|}{\bf Heading1} & \multicolumn{4}{|l|}{\bf Heading2}\\ \thickhline
$cell1 row1$ & cell2 row 1 & cell3 row 1 & cell4 row 1 & cell5 row 1 & cell6 row 1 & cell7 row 1 & cell8 row 1\\ \hline
$cell1 row2$ & cell2 row 2 & cell3 row 2 & cell4 row 2 & cell5 row 2 & cell6 row 2 & cell7 row 2 & cell8 row 2\\ \hline
$cell1 row3$ & cell2 row 3 & cell3 row 3 & cell4 row 3 & cell5 row 3 & cell6 row 3 & cell7 row 3 & cell8 row 3\\ \hline
\end{tabular}
\begin{flushleft} Table notes Phasellus venenatis, tortor nec vestibulum mattis, massa tortor interdum felis, nec pellentesque metus tortor nec nisl. Ut ornare mauris tellus, vel dapibus arcu suscipit sed.
\end{flushleft}
\label{table1}
\end{adjustwidth}
\end{table}


%PLOS does not support heading levels beyond the 3rd (no 4th level headings).
\subsection*{\lorem\ and \ipsum\ Nunc blandit a tortor.}
\subsubsection*{3rd Level Heading.} 
Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque. Quisque augue sem, tincidunt sit amet feugiat eget, ullamcorper sed velit. Sed non aliquet felis. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris commodo justo ac dui pretium imperdiet. Sed suscipit iaculis mi at feugiat. 

\begin{enumerate}
	\item{react}
	\item{diffuse free particles}
	\item{increment time by dt and go to 1}
\end{enumerate}


\subsection*{Sed ac quam id nisi malesuada congue.}

Nulla mi mi, venenatis sed ipsum varius, volutpat euismod diam. Proin rutrum vel massa non gravida. Quisque tempor sem et dignissim rutrum. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi at justo vitae nulla elementum commodo eu id massa. In vitae diam ac augue semper tincidunt eu ut eros. Fusce fringilla erat porttitor lectus cursus, vel sagittis arcu lobortis. Aliquam in enim semper, aliquam massa id, cursus neque. Praesent faucibus semper libero.

\begin{itemize}
	\item First bulleted item.
	\item Second bulleted item.
	\item Third bulleted item.
\end{itemize}

\section*{Discussion}
Nulla mi mi, venenatis sed ipsum varius, Table~\ref{table1} volutpat euismod diam. Proin rutrum vel massa non gravida. Quisque tempor sem et dignissim rutrum. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi at justo vitae nulla elementum commodo eu id massa. In vitae diam ac augue semper tincidunt eu ut eros. Fusce fringilla erat porttitor lectus cursus, vel sagittis arcu lobortis. Aliquam in enim semper, aliquam massa id, cursus neque. Praesent faucibus semper libero~\cite{bib3}.

\section*{Conclusion}

CO$_2$ Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque. Quisque augue sem, tincidunt sit amet feugiat eget, ullamcorper sed velit. 

Sed non aliquet felis. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris commodo justo ac dui pretium imperdiet. Sed suscipit iaculis mi at feugiat. Ut neque ipsum, luctus id lacus ut, laoreet scelerisque urna. Phasellus venenatis, tortor nec vestibulum mattis, massa tortor interdum felis, nec pellentesque metus tortor nec nisl. Ut ornare mauris tellus, vel dapibus arcu suscipit sed. Nam condimentum sem eget mollis euismod. Nullam dui urna, gravida venenatis dui et, tincidunt sodales ex. Nunc est dui, sodales sed mauris nec, auctor sagittis leo. Aliquam tincidunt, ex in facilisis elementum, libero lectus luctus est, non vulputate nisl augue at dolor. For more information, see \nameref{S1_Appendix}.

\section*{Supporting Information}

% Include only the SI item label in the paragraph heading. Use the \nameref{label} command to cite SI items in the text.
\paragraph*{S1 Fig.}
\label{S1_Fig}
{\bf Bold the title sentence.} Add descriptive text after the title of the item (optional).

\paragraph*{S2 Fig.}
\label{S2_Fig}
{\bf Lorem Ipsum.} Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

\paragraph*{S1 File.}
\label{S1_File}
{\bf Lorem Ipsum.}  Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

\paragraph*{S1 Video.}
\label{S1_Video}
{\bf Lorem Ipsum.}  Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

\paragraph*{S1 Appendix.}
\label{S1_Appendix}
{\bf Lorem Ipsum.} Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

\paragraph*{S1 Table.}
\label{S1_Table}
{\bf Lorem Ipsum.} Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

\section*{Acknowledgments}
Cras egestas velit mauris, eu mollis turpis pellentesque sit amet. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam id pretium nisi. Sed ac quam id nisi malesuada congue. Sed interdum aliquet augue, at pellentesque quam rhoncus vitae.

\nolinenumbers

% Either type in your references using
% \begin{thebibliography}{}
% \bibitem{}
% Text
% \end{thebibliography}
%
% or
%
% Compile your BiBTeX database using our plos2015.bst
% style file and paste the contents of your .bbl file
% here.
% 
%\begin{thebibliography}{10}
%
%\bibitem{bib1}

%Conant GC, Wolfe KH.
%\newblock {{T}urning a hobby into a job: how duplicated genes find new
%  functions}.
%\newblock Nat Rev Genet. 2008 Dec;9(12):938--950.

%\end{thebibliography}

%REMEMBER TO FIX UP THESE CITATIONS AND USE THEIR FORMATTING ONCE I HAVE A MORE COMPLEATE VERSION.

%\bibliographystyle{/home/shauncoutts/Dropbox/shauns_paper/referencing/bes} 
\bibliography{/home/shauncoutts/Dropbox/shauns_paper/referencing/refs}


\end{document}

